telegram在vs2022中编译失败

### Steps to reproduce

正常编译

### Expected behaviour

无错误

### Actual behaviour

![image](https://github.com/telegramdesktop/tdesktop/assets/50785665/e336290c-487a-459f-b9ce-99f3d6d5ab3b)
提示无法找到emoji.h

### Operating system

windows10

### Version of Telegram Desktop

dev版本

### Installation source

Static binary from official website

### Crash ID

_No response_

### Logs

_No response_

已启动重新生成...
1>------ 已启动全部重新生成: 项目: ZERO_CHECK, 配置: Debug Win32 ------
1>1>Checking Build System
2>------ 已启动全部重新生成: 项目: win_directx_helper, 配置: Debug Win32 ------
3>------ 已启动全部重新生成: 项目: td_scheme_scheme, 配置: Debug Win32 ------
4>------ 已启动全部重新生成: 项目: lib_tgvoip_bundled, 配置: Debug Win32 ------
5>------ 已启动全部重新生成: 项目: lib_tgcalls, 配置: Debug Win32 ------
3>Generating scheme (td_scheme)
2>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/win_directx_helper/CMakeLists.txt
5>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
4>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
2>win_directx_helper.cpp
5>Instance.cpp
4>BlockingQueue.cpp
5>AudioDeviceHelper.cpp
5>ChannelManager.cpp
4>Buffers.cpp
4>CongestionControl.cpp
4>EchoCanceller.cpp
5>CodecSelectHelper.cpp
4>JitterBuffer.cpp
5>CryptoHelper.cpp
4>logging.cpp
5>EncryptedConnection.cpp
4>MediaStreamItf.cpp
3>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
5>FakeAudioDeviceModule.cpp
4>OpusDecoder.cpp
4>OpusEncoder.cpp
4>VoIPController.cpp
5>FieldTrialsConfig.cpp
4>VoIPGroupController.cpp
2>win_directx_helper.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\win_directx_helper\Debug\libwin_directx_helper.a
4>VoIPServerConfig.cpp
5>InstanceImpl.cpp
4>AudioInput.cpp
5>LogSinkImpl.cpp
5>Manager.cpp
5>MediaManager.cpp
4>AudioOutput.cpp
6>------ 已启动全部重新生成: 项目: lib_crl, 配置: Debug Win32 ------
5>Message.cpp
4>Resampler.cpp
4>NetworkSocket.cpp
4>PacketReassembler.cpp
4>MessageThread.cpp
4>AudioIO.cpp
4>ScreamCongestionController.cpp
4>VideoSource.cpp
4>VideoRenderer.cpp
4>json11.cpp
7>------ 已启动全部重新生成: 项目: external_kimageformats_init, 配置: Debug Win32 ------
4>NetworkSocketWinsock.cpp
4>AudioInputWave.cpp
4>AudioOutputWave.cpp
5>NetworkManager.cpp
5>SctpDataChannelProviderInterfaceImpl.cpp
6>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_crl/CMakeLists.txt
4>AudioOutputWASAPI.cpp
5>StaticThreads.cpp
5>TurnCustomizerImpl.cpp
4>AudioInputWASAPI.cpp
4>WindowsSpecific.cpp
5>VideoCaptureInterface.cpp
5>VideoCaptureInterfaceImpl.cpp
5>gzip.cpp
5>ContentNegotiation.cpp
6>crl_common_fp_exceptions.cpp
7>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/kimageformats/CMakeLists.txt
6>crl_common_list.cpp
6>crl_common_on_main.cpp
6>crl_common_queue.cpp
6>crl_dispatch_async.cpp
6>crl_dispatch_queue.cpp
6>crl_dispatch_semaphore.cpp
6>crl_qt_async.cpp
6>crl_qt_semaphore.cpp
6>crl_winapi_async.cpp
6>crl_winapi_fp_exceptions.cpp
6>crl_winapi_list.cpp
6>crl_winapi_semaphore.cpp
6>crl_winapi_time.cpp
4>lib_tgvoip_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\Debug\liblib_tgvoip_bundled.a
5>DirectNetworkingImpl.cpp
5>ExternalSignalingConnection.cpp
7>init.cpp
5>InstanceV2ReferenceImpl.cpp
6>crl_object_on_thread.cpp
5>InstanceV2Impl.cpp
6>crl_time.cpp
8>------ 已启动全部重新生成: 项目: external_kimageformats, 配置: Debug Win32 ------
5>NativeNetworkingImpl.cpp
6>lib_crl.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_crl\lib_crl.dir\Debug\lib_crl.lib
7>external_kimageformats_init.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\kimageformats\external_kimageformats_init.dir\Debug\external_kimageformats_init.lib
9>------ 已启动全部重新生成: 项目: lib_base, 配置: Debug Win32 ------
5>ReflectorPort.cpp
10>------ 已启动全部重新生成: 项目: external_kcoreaddons, 配置: Debug Win32 ------
9>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_base/CMakeLists.txt
10>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/kcoreaddons/CMakeLists.txt
9>cmake_pch.cxx
5>ReflectorRelayPortFactory.cpp
5>Signaling.cpp
5>SignalingConnection.cpp
5>SignalingEncryption.cpp
5>SignalingSctpConnection.cpp
10>kurlmimedata.cpp
10>ksandbox.cpp
5>InstanceV2_4_0_0Impl.cpp
5>Signaling_4_0_0.cpp
5>DesktopCaptureSource.cpp
5>DesktopCaptureSourceHelper.cpp
8>Generating __/__/__/Telegram/ThirdParty/kimageformats/src/imageformats/moc_avif_p.cpp
8>Generating __/__/__/Telegram/ThirdParty/kimageformats/src/imageformats/moc_heif_p.cpp
5>DesktopCaptureSourceManager.cpp
5>AVIOContextImpl.cpp
10>external_kcoreaddons.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\kcoreaddons\Debug\libexternal_kcoreaddons.a
9>base_battery_saving_win.cpp
9>base_file_utilities_win.cpp
9>base_global_shortcuts_win.cpp
9>base_info_win.cpp
8>Generating __/__/__/Telegram/ThirdParty/kimageformats/src/imageformats/moc_jxl_p.cpp
5>AudioStreamingPart.cpp
9>base_last_input_win.cpp
5>AudioStreamingPartInternal.cpp
11>------ 已启动全部重新生成: 项目: codegen_common, 配置: Debug Win32 ------
9>base_layout_switch_win.cpp
9>base_network_reachability_win.cpp
5>AudioStreamingPartPersistentDecoder.cpp
9>base_power_save_blocker_win.cpp
9>base_process_win.cpp
9>base_system_media_controls_win.cpp
5>GroupInstanceCustomImpl.cpp
9>base_url_scheme_win.cpp
9>base_windows_safe_library.cpp
5>GroupJoinPayloadInternal.cpp
9>base_windows_winrt.cpp
5>GroupNetworkManager.cpp
5>StreamingMediaContext.cpp
8>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/kimageformats/CMakeLists.txt
9>base_windows_wrl.cpp
9>base_platform_info.cpp
9>algorithm.cpp
9>assertion.cpp
5>VideoStreamingPart.cpp
9>base_file_utilities.cpp
9>battery_saving.cpp
9>bytes.cpp
11>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/common/CMakeLists.txt
9>call_delayed.cpp
9>crc32hash.cpp
9>concurrent_timer.cpp
9>debug_log.cpp
9>event_filter.cpp
9>file_lock_win.cpp
9>global_shortcuts_generic.cpp
9>integration.cpp
9>last_user_input.cpp
9>network_reachability.cpp
9>options.cpp
9>parse_helper.cpp
9>power_save_blocker.cpp
9>qthelp_url.cpp
9>random.cpp
9>runtime_composer.cpp
9>single_instance.cpp
9>timer.cpp
9>unixtime.cpp
5>DesktopInterface.cpp
9>qt_tab_key.cpp
5>VideoCapturerInterfaceImpl.cpp
8>avif.cpp
8>heif.cpp
8>jxl.cpp
8>moc_avif_p.cpp
5>VideoCapturerTrackSource.cpp
11>basic_tokenized_file.cpp
11>checked_utf8_string.cpp
11>clean_file.cpp
11>cpp_file.cpp
11>logging.cpp
9>lib_base.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_base\Debug\liblib_base.a
8>moc_heif_p.cpp
8>moc_jxl_p.cpp
5>VideoCameraCapturer.cpp
5>json11.cpp
12>------ 已启动全部重新生成: 项目: lib_tl, 配置: Debug Win32 ------
11>codegen_common.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\codegen\codegen\common\codegen_common.dir\Debug\codegen_common.lib
13>------ 已启动全部重新生成: 项目: codegen_style, 配置: Debug Win32 ------
13>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/style/CMakeLists.txt
13>generator.cpp
13>main.cpp
13>module.cpp
13>options.cpp
13>parsed_file.cpp
12>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_tl/CMakeLists.txt
13>processor.cpp
13>structure_types.cpp
12>tl_basic_types.cpp
5>lib_tgcalls.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\Debug\liblib_tgcalls.a
14>------ 已启动全部重新生成: 项目: lib_tgcalls_legacy, 配置: Debug Win32 ------
8>external_kimageformats.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\kimageformats\Debug\libexternal_kimageformats.a
15>------ 已启动全部重新生成: 项目: external_qt_static_plugins, 配置: Debug Win32 ------
12>lib_tl.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_tl\lib_tl.dir\Debug\lib_tl.lib
16>------ 已启动全部重新生成: 项目: td_scheme, 配置: Debug Win32 ------
15>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/qt/qt_static_plugins/CMakeLists.txt
14>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
15>qt_static_plugins.cpp
16>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
13>LINK : fatal error LNK1104: 无法打开文件“dxguid.obj”
13>已完成生成项目“codegen_style.vcxproj”的操作 - 失败。
17>------ 已启动全部重新生成: 项目: td_ui_styles, 配置: Debug Win32 ------
16>scheme.cpp
16>scheme-dump_to_text.cpp
17>Generating styles (td_ui)
17>'codegen\codegen\style\Debug\codegen_style.exe' 不是内部或外部命令，也不是可运行的程序
17>或批处理文件。
17>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\5d71192b9cd298b8cb10a9827e4946ce\td_ui_style.timestamp.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\868136f9c1deeb6abe216f3361c29fd3\td_ui_styles.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\CMakeLists.txt”的自定义生成已退出，代码为 9009。
17>已完成生成项目“td_ui_styles.vcxproj”的操作 - 失败。
18>------ 已启动全部重新生成: 项目: lib_ui_styles, 配置: Debug Win32 ------
18>Generating styles (lib_ui)
18>'..\codegen\codegen\style\Debug\codegen_style.exe' 不是内部或外部命令，也不是可运行的程序
18>或批处理文件。
18>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\14900836e4a4ec0e1ba31f97b7dd2f36\lib_ui_style.timestamp.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3b7f912706227bff2b098755786bcfe6\lib_ui_styles.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\CMakeLists.txt”的自定义生成已退出，代码为 9009。
18>已完成生成项目“lib_ui_styles.vcxproj”的操作 - 失败。
19>------ 已启动全部重新生成: 项目: lib_ui_palette, 配置: Debug Win32 ------
19>Generating palette (lib_ui)
19>'..\codegen\codegen\style\Debug\codegen_style.exe' 不是内部或外部命令，也不是可运行的程序
19>或批处理文件。
19>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\colors.palette;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3b7f912706227bff2b098755786bcfe6\lib_ui_palette.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\CMakeLists.txt”的自定义生成已退出，代码为 9009。
19>已完成生成项目“lib_ui_palette.vcxproj”的操作 - 失败。
20>------ 已启动全部重新生成: 项目: codegen_lang, 配置: Debug Win32 ------
20>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/lang/CMakeLists.txt
20>generator.cpp
20>main.cpp
20>options.cpp
20>parsed_file.cpp
14>InstanceImplLegacy.cpp
15>external_qt_static_plugins.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\qt\qt_static_plugins\external_qt_static_plugins.dir\Debug\external_qt_static_plugins.lib
20>processor.cpp
21>------ 已启动全部重新生成: 项目: codegen_emoji, 配置: Debug Win32 ------
14>lib_tgcalls_legacy.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\Debug\liblib_tgcalls_legacy.a
21>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/emoji/CMakeLists.txt
22>------ 已启动全部重新生成: 项目: lib_stripe, 配置: Debug Win32 ------
21>data.cpp
21>data_old.cpp
21>data_read.cpp
21>generator.cpp
22>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
20>LINK : fatal error LNK1104: 无法打开文件“dxguid.obj”
21>main.cpp
20>已完成生成项目“codegen_lang.vcxproj”的操作 - 失败。
23>------ 已启动全部重新生成: 项目: td_lang_lang, 配置: Debug Win32 ------
21>options.cpp
22>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): warning MSB8065: 项“C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\CMakeLists.txt”的自定义生成成功，但尚未创建指定的输出“c:\users\admin\desktop\tbuild\buildpath\tdesktop\out\telegram\cmakefiles\generate.stamp”。这可能会导致增量生成无法正常工作。
21>replaces.cpp
22>cmake_pch.cxx
23>Generating lang (td_lang)
23>'codegen\codegen\lang\Debug\codegen_lang.exe' 不是内部或外部命令，也不是可运行的程序
23>或批处理文件。
23>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\a8ad2d315d3f78d762dc15d12a0bb9e3\lang_auto.timestamp.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\868136f9c1deeb6abe216f3361c29fd3\td_lang_lang.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\CMakeLists.txt”的自定义生成已退出，代码为 9009。
23>已完成生成项目“td_lang_lang.vcxproj”的操作 - 失败。
24>------ 已启动全部重新生成: 项目: lib_storage, 配置: Debug Win32 ------
24>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_storage/CMakeLists.txt
24>cmake_pch.cxx
16>td_scheme.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\td_scheme.dir\Debug\td_scheme.lib
21>LINK : fatal error LNK1104: 无法打开文件“dxguid.obj”
22>stripe_api_client.cpp
22>stripe_card.cpp
22>stripe_card_params.cpp
22>stripe_card_validator.cpp
21>已完成生成项目“codegen_emoji.vcxproj”的操作 - 失败。
25>------ 已启动全部重新生成: 项目: lib_ui_emoji, 配置: Debug Win32 ------
22>stripe_decode.cpp
22>stripe_error.cpp
22>stripe_form_encoder.cpp
22>stripe_token.cpp
22>smartglocal_api_client.cpp
22>smartglocal_card.cpp
22>smartglocal_error.cpp
22>smartglocal_token.cpp
25>Generating emoji (lib_ui)
25>'..\codegen\codegen\emoji\Debug\codegen_emoji.exe' 不是内部或外部命令，也不是可运行的程序
25>或批处理文件。
25>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3c095a66caa0e85d9452707996a8760a\emoji.timestamp.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3b7f912706227bff2b098755786bcfe6\lib_ui_emoji.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\CMakeLists.txt”的自定义生成已退出，代码为 9009。
25>已完成生成项目“lib_ui_emoji.vcxproj”的操作 - 失败。
26>------ 已启动全部重新生成: 项目: lib_ui_autogen, 配置: Debug Win32 ------
22>lib_stripe.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_stripe.dir\Debug\lib_stripe.lib
22>已完成生成项目“lib_stripe.vcxproj”的操作。
27>------ 已启动全部重新生成: 项目: external_rlottie_bundled, 配置: Debug Win32 ------
27>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/rlottie/CMakeLists.txt
27>lottieanimation.cpp
27>lottieitem.cpp
27>lottiekeypath.cpp
27>lottieloader.cpp
27>lottiemodel.cpp
27>lottieparser.cpp
27>lottieproxymodel.cpp
27>v_ft_math.cpp
27>v_ft_raster.cpp
27>v_ft_stroker.cpp
27>vregion.cpp
27>vbezier.cpp
27>vbitmap.cpp
28>------ 已启动全部重新生成: 项目: td_mtproto, 配置: Debug Win32 ------
24>storage_clear_legacy.cpp
27>vbrush.cpp
24>storage_clear_legacy_win.cpp
24>storage_databases.cpp
27>vcompositionfunctions.cpp
24>storage_encrypted_file.cpp
27>vdasher.cpp
24>storage_encryption.cpp
24>storage_cache_binlog_reader.cpp
24>storage_cache_cleaner.cpp
24>storage_cache_compactor.cpp
27>vdebug.cpp
24>storage_cache_database.cpp
27>vdrawable.cpp
27>vdrawhelper.cpp
27>vdrawhelper_neon.cpp
24>storage_cache_database_object.cpp
27>vdrawhelper_sse2.cpp
27>velapsedtimer.cpp
24>storage_cache_types.cpp
27>vimageloader.cpp
27>vinterpolator.cpp
27>vmatrix.cpp
27>vpainter.cpp
27>vpath.cpp
27>vpathmesure.cpp
27>vraster.cpp
26>1>Automatic MOC for target lib_ui
28>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
27>vrect.cpp
27>vrle.cpp
26>
24>lib_storage.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_storage\lib_storage.dir\Debug\lib_storage.lib
26>AutoGen error
26>-------------
26>Info error in info file
26>"C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/out/Telegram/lib_ui/CMakeFiles/lib_ui_autogen.dir/AutogenInfo.json":
26>The header file "SRC:/out/Telegram/lib_ui/gen/emoji.h" does not exist.
26>
29>------ 已启动全部重新生成: 项目: td_export, 配置: Debug Win32 ------
26>系统找不到指定的批处理标签 - VCEnd
28>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): warning MSB8065: 项“C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\CMakeLists.txt”的自定义生成成功，但尚未创建指定的输出“c:\users\admin\desktop\tbuild\buildpath\tdesktop\out\telegram\cmakefiles\generate.stamp”。这可能会导致增量生成无法正常工作。
26>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(232,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3b7f912706227bff2b098755786bcfe6\lib_ui_autogen.rule”的自定义生成已退出，代码为 1。
26>已完成生成项目“lib_ui_autogen.vcxproj”的操作 - 失败。
30>------ 已启动全部重新生成: 项目: external_qr_code_generator_bundled, 配置: Debug Win32 ------
28>cmake_pch.cxx
30>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/qr_code_generator/CMakeLists.txt
29>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
30>qrcodegen.cpp
29>cmake_pch.cxx
27>external_rlottie_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\rlottie\Debug\libexternal_rlottie_bundled.a
31>------ 已启动全部重新生成: 项目: external_minizip_bundled, 配置: Debug Win32 ------
31>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/minizip/CMakeLists.txt
31>ioapi.c
31>zip.c
31>unzip.c
30>external_qr_code_generator_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\qr_code_generator\Debug\libexternal_qr_code_generator_bundled.a
32>------ 已启动全部重新生成: 项目: lib_qr, 配置: Debug Win32 ------
31>external_minizip_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\minizip\Debug\libexternal_minizip_bundled.a
33>------ 已启动全部重新生成: 项目: external_lz4_bundled, 配置: Debug Win32 ------
32>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_qr/CMakeLists.txt
32>qr_generate.cpp
33>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/lz4/CMakeLists.txt
33>lz4.c
33>lz4frame.c
33>lz4hc.c
33>xxhash.c
32>lib_qr.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_qr\lib_qr.dir\Debug\lib_qr.lib
33>external_lz4_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\lz4\Debug\libexternal_lz4_bundled.a
34>------ 已启动全部重新生成: 项目: external_hunspell_bundled, 配置: Debug Win32 ------
35>------ 已启动全部重新生成: 项目: lib_ui, 配置: Debug Win32 ------
34>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/hunspell/CMakeLists.txt
34>affentry.cxx
34>affixmgr.cxx
34>csutil.cxx
34>filemgr.cxx
34>hashmgr.cxx
34>hunspell.cxx
34>hunzip.cxx
34>phonet.cxx
34>replist.cxx
34>suggestmgr.cxx
29>export_api_wrap.cpp
28>mtproto_abstract_socket.cpp
29>export_controller.cpp
29>export_settings.cpp
29>export_data_types.cpp
28>mtproto_bound_key_creator.cpp
28>mtproto_dc_key_binder.cpp
28>mtproto_dc_key_creator.cpp
29>export_output_abstract.cpp
28>mtproto_dcenter.cpp
28>mtproto_domain_resolver.cpp
28>mtproto_dump_to_text.cpp
28>mtproto_received_ids_manager.cpp
28>mtproto_rsa_public_key.cpp
28>mtproto_serialized_request.cpp
28>mtproto_tcp_socket.cpp
28>mtproto_tls_socket.cpp
28>mtproto_auth_key.cpp
29>export_output_file.cpp
28>mtproto_concurrent_sender.cpp
34>external_hunspell_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\hunspell\Debug\libexternal_hunspell_bundled.a
28>mtproto_config.cpp
28>mtproto_dc_options.cpp
29>export_output_html.cpp
28>mtproto_dh_utils.cpp
28>mtproto_proxy_data.cpp
36>------ 已启动全部重新生成: 项目: external_cld3_bundled_task_spec.proto, 配置: Debug Win32 ------
28>mtproto_response.cpp
29>export_output_json.cpp
35>1>Automatic RCC for fonts/fonts.qrc
36>Generating protobuf task_spec.proto (external_cld3_bundled)
29>export_output_stats.cpp
35>2>Automatic RCC for qt_conf/win.qrc
28>td_mtproto.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\td_mtproto.dir\Debug\td_mtproto.lib
36>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
35>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_ui/CMakeLists.txt
37>------ 已启动全部重新生成: 项目: external_cld3_bundled_sentence.proto, 配置: Debug Win32 ------
28>已完成生成项目“td_mtproto.vcxproj”的操作。
37>Generating protobuf sentence.proto (external_cld3_bundled)
38>------ 已启动全部重新生成: 项目: external_cld3_bundled_feature_extractor.proto, 配置: Debug Win32 ------
35>cmake_pch.cxx
38>Generating protobuf feature_extractor.proto (external_cld3_bundled)
37>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
38>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
39>------ 已启动全部重新生成: 项目: Telegram_midl, 配置: Debug Win32 ------
39>Generating headers from IDLs (Telegram)
40>------ 已启动全部重新生成: 项目: external_cld3_bundled, 配置: Debug Win32 ------
39>Microsoft (R) 32b/64b MIDL Compiler Version 8.01.0626
39>Copyright (c) Microsoft Corporation. All rights reserved.
39>Processing C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\platform\win\windows_quiethours.idl
39>windows_quiethours.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\oaidl.idl
39>oaidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\objidl.idl
39>objidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\unknwn.idl
39>unknwn.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\wtypes.idl
39>wtypes.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\wtypesbase.idl
39>wtypesbase.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\basetsd.h
39>basetsd.h
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\guiddef.h
39>guiddef.h
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\oaidl.acf
39>oaidl.acf
39>Microsoft (R) 32b/64b MIDL Compiler Version 8.01.0626
39>Copyright (c) Microsoft Corporation. All rights reserved.
39>Processing C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\platform\win\windows_toastactivator.idl
39>windows_toastactivator.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\oaidl.idl
39>oaidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\objidl.idl
39>objidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\unknwn.idl
39>unknwn.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\wtypes.idl
39>wtypes.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\wtypesbase.idl
39>wtypesbase.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\basetsd.h
39>basetsd.h
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\guiddef.h
39>guiddef.h
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\ocidl.idl
39>ocidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\oleidl.idl
39>oleidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\servprov.idl
39>servprov.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\urlmon.idl
39>urlmon.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\msxml.idl
39>msxml.idl
29>td_export.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\td_export.dir\Debug\td_export.lib
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\oaidl.acf
39>oaidl.acf
40>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\ocidl.acf
39>ocidl.acf
39>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
40>feature_extractor.pb.cc
40>sentence.pb.cc
40>task_spec.pb.cc
40>base.cc
40>embedding_feature_extractor.cc
40>embedding_network.cc
40>feature_extractor.cc
40>feature_types.cc
40>fml_parser.cc
40>language_identifier_features.cc
35>mocs_compilation_Debug.cpp
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\lib_ui_autogen\mocs_compilation_Debug.cpp”: No such file or directory
35>palette.cpp
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\palette.cpp”: No such file or directory
35>style_basic.cpp
40>lang_id_nn_params.cc
40>nnet_language_identifier.cc
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\style_basic.cpp”: No such file or directory
35>style_layers.cpp
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\style_layers.cpp”: No such file or directory
35>style_widgets.cpp
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\style_widgets.cpp”: No such file or directory
35>emoji.cpp
35>emoji_suggestions_data.cpp
35>dpr_icon.cpp
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\emoji.cpp”: No such file or directory
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\emoji_suggestions_data.cpp”: No such file or directory
35>animation_value.cpp
35>animations.cpp
35>cross_animation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/cross_animation.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\cross_animation.cpp)
35>cross_line.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/cross_line.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\cross_line.cpp)
35>fade_animation.cpp
35>frame_generator.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\animation_value.cpp)
35>gradient.cpp
40>registry.cc
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/fade_animation.h(11,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\fade_animation.cpp)
35>numbers_animation.cpp
40>relevant_script_feature.cc
35>panel_animation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/panel_animation.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\panel_animation.cpp)
35>path_shift_gradient.cpp
35>radial_animation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\numbers_animation.cpp)
35>ripple_animation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/ripple_animation.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\ripple_animation.cpp)
40>sentence_features.cc
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\radial_animation.cpp)
35>round_area_with_shadow.cpp
35>show_animation.cpp
35>slide_animation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\round_area_with_shadow.cpp)
35>spoiler_mess.cpp
35>gl_detection.cpp
35>gl_image.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\spoiler_mess.cpp)
35>gl_math.cpp
40>task_context.cc
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\show_animation.cpp(11,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory
35>gl_primitives.cpp
35>gl_shader.cpp
40>task_context_params.cc
40>unicodetext.cc
35>gl_surface.cpp
40>utils.cc
35>gl_window.cpp
35>image_prepare.cpp
40>workspace.cc
35>box_content.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\image\image_prepare.cpp)
35>box_layer_widget.cpp
40>generated_entities.cc
40>getonescriptspan.cc
40>utf8statetable.cc
40>offsetmap.cc
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\box_content.cpp)
35>generic_box.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\gl\gl_surface.cpp)
35>layer_manager.cpp
40>text_processing.cc
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\box_layer_widget.cpp)
40>fixunicodevalue.cc
35>layer_widget.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\generic_box.cpp)
35>show.cpp
35>arcs.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\layer_widget.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/paint/arcs.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\arcs.cpp)
35>blob.cpp
35>blobs.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\blob.cpp)
35>blobs_linear.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\blobs.cpp)
35>ui_window_shadow_win.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\blobs_linear.cpp)
35>ui_window_title_win.cpp
40>external_cld3_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\cld3\Debug\libexternal_cld3_bundled.a
35>ui_window_win.cpp
35>ui_windows_direct_manipulation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_window_shadow_win.cpp(12,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory
35>ui_utility_win.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_window_title_win.cpp)
35>ui_platform_window_title.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_window_win.cpp(19,1): fatal  error C1083: 无法打开包括文件: “styles/palette.h”: No such file or directory
35>ui_platform_window.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\ui_platform_window_title.cpp)
35>style_core.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/popup_menu.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_utility_win.cpp)
35>style_core_color.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/style/style_core_palette.h(10,1): fatal  error C1083: 无法打开包括文件: “styles/palette.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core_color.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core.cpp)
35>style_core_custom_font.cpp
35>style_core_direction.cpp
35>style_core_font.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\ui_platform_window.cpp)
35>style_core_icon.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/style/style_core_palette.h(10,1): fatal  error C1083: 无法打开包括文件: “styles/palette.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core_icon.cpp)
35>style_core_palette.cpp
35>style_core_scale.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/style/style_core_palette.h(10,1): fatal  error C1083: 无法打开包括文件: “styles/palette.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core_palette.cpp)
35>style_core_types.cpp
35>style_palette_colorizer.cpp
35>custom_emoji_instance.cpp
35>text.cpp
35>text_block.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_block.cpp)
35>text_custom_emoji.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text.cpp)
35>text_entity.cpp
35>text_parser.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_parser.cpp)
35>text_renderer.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\custom_emoji_instance.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_entity.cpp)
35>text_spoiler_data.cpp
35>text_utilities.cpp
35>text_variant.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_renderer.cpp)
35>toast.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_spoiler_data.cpp)
35>toast_manager.cpp
35>toast_widget.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\toast\toast.cpp)
35>box_content_divider.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\toast\toast_widget.cpp)
35>buttons.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\toast\toast_manager.cpp)
35>call_button.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\box_content_divider.cpp(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_layers.h”: No such file or directory
35>checkbox.cpp
35>dropdown_menu.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/dropdown_menu.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\dropdown_menu.cpp)
35>icon_button_with_text.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\buttons.cpp)
35>inner_dropdown.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/inner_dropdown.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\inner_dropdown.cpp)
35>input_fields.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\input_fields.cpp)
35>labels.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\call_button.cpp)
35>time_part_input.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\fields\time_part_input.cpp)
35>menu.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\checkbox.cpp)
35>menu_action.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\icon_button_with_text.cpp)
35>menu_add_action_callback.cpp
35>menu_add_action_callback_factory.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_action.cpp)
35>menu_common.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_add_action_callback_factory.cpp)
35>menu_item_base.cpp
35>menu_multiline_action.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\labels.cpp)
35>menu_separator.cpp
35>menu_toggle.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_toggle.cpp)
35>elastic_scroll.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_multiline_action.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_item_base.cpp)
35>popup_menu.cpp
35>rp_window.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/popup_menu.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\popup_menu.cpp)
35>separate_panel.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_separator.cpp)
35>scroll_area.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\separate_panel.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/scroll_area.h(13,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\elastic_scroll.cpp)
35>side_bar_button.cpp
35>shadow.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/scroll_area.h(13,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\scroll_area.cpp)
35>time_input.cpp
35>tooltip.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\shadow.cpp(10,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory
35>fade_wrap.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\side_bar_button.cpp)
35>follow_slide_wrap.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\tooltip.cpp)
35>padding_wrap.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/fade_animation.h(11,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\fade_wrap.cpp)
35>slide_wrap.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\time_input.cpp)
35>vertical_layout.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\follow_slide_wrap.cpp(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory
35>vertical_layout_reorder.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\slide_wrap.cpp(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory
35>abstract_button.cpp
35>animated_icon.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/scroll_area.h(13,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\vertical_layout_reorder.cpp)
35>basic_click_handlers.cpp
35>cached_special_layer_shadow_corners.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/ripple_animation.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\cached_special_layer_shadow_corners.cpp)
35>click_handler.cpp
35>delayed_activation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\basic_click_handlers.cpp)
35>dragging_scroll_manager.cpp
35>emoji_config.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\emoji_config.cpp)
35>inactive_press.cpp
35>integration.cpp
35>main_queue_processor.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\integration.cpp)
35>ph.cpp
35>round_rect.cpp
35>rp_widget.cpp
35>ui_utility.cpp
35>emoji_suggestions.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\emoji_suggestions\emoji_suggestions.cpp(10,1): fatal  error C1083: 无法打开包括文件: “emoji_suggestions_data.h”: No such file or directory
35>qrc_win.cpp
35>qrc_fonts.cpp
35>已完成生成项目“lib_ui.vcxproj”的操作 - 失败。
41>------ 已启动全部重新生成: 项目: td_lang, 配置: Debug Win32 ------
42>------ 已启动全部重新生成: 项目: lib_webview, 配置: Debug Win32 ------
43>------ 已启动全部重新生成: 项目: lib_spellcheck, 配置: Debug Win32 ------
44>------ 已启动全部重新生成: 项目: lib_ffmpeg, 配置: Debug Win32 ------
43>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_spellcheck/CMakeLists.txt
42>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_webview/CMakeLists.txt
44>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
41>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
42>webview_dialog.cpp
43>cmake_pch.cxx
44>ffmpeg_frame_generator.cpp
42>webview_embed.cpp
42>webview_win.cpp
44>ffmpeg_utility.cpp
42>webview_windows_edge_chromium.cpp
42>webview_windows_edge_html.cpp
41>cmake_pch.cxx
42>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_webview\webview\webview_dialog.cpp)
44>lib_ffmpeg.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ffmpeg.dir\Debug\lib_ffmpeg.lib
43>spellcheck_win.cpp
43>language_cld3.cpp
43>hunspell_controller.cpp
43>spellcheck_utils.cpp
41>lang_auto.cpp
41>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\gen\lang_auto.cpp”: No such file or directory
41>lang_file_parser.cpp
41>lang_keys.cpp
41>lang_tag.cpp
41>lang_text_entity.cpp
41>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang/lang_keys.h(10,1): fatal  error C1083: 无法打开包括文件: “lang_auto.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang\lang_keys.cpp)
41>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang/lang_keys.h(10,1): fatal  error C1083: 无法打开包括文件: “lang_auto.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang\lang_file_parser.cpp)
41>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang/lang_keys.h(10,1): fatal  error C1083: 无法打开包括文件: “lang_auto.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang\lang_tag.cpp)
43>spelling_highlighter.cpp
43>spelling_highlighter_helper.cpp
43>spellcheck_value.cpp
43>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_spellcheck\spellcheck\spelling_highlighter.cpp)
41>已完成生成项目“td_lang.vcxproj”的操作 - 失败。
43>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_spellcheck\spellcheck\spelling_highlighter_helper.cpp)
42>已完成生成项目“lib_webview.vcxproj”的操作 - 失败。
45>------ 已启动全部重新生成: 项目: lib_webrtc, 配置: Debug Win32 ------
46>------ 已启动全部重新生成: 项目: lib_lottie, 配置: Debug Win32 ------
46>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_lottie/CMakeLists.txt
45>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_webrtc/CMakeLists.txt
45>cmake_pch.cxx
46>lottie_frame_provider_direct.cpp
46>lottie_frame_renderer.cpp
46>lottie_animation.cpp
46>lottie_common.cpp
46>lottie_frame_generator.cpp
43>已完成生成项目“lib_spellcheck.vcxproj”的操作 - 失败。
46>lottie_multi_player.cpp
46>lottie_single_player.cpp
46>lottie_icon.cpp
45>webrtc_audio_input_tester.cpp
45>webrtc_create_adm.cpp
45>webrtc_media_devices.cpp
45>webrtc_video_track.cpp
45>webrtc_openal_adm.cpp
46>lottie_cache.cpp
45>webrtc_loopback_adm_win.cpp
46>lottie_cache_frame_storage.cpp
46>lottie_frame_provider_cached.cpp
46>lottie_frame_provider_cached_multi.cpp
46>lottie_frame_provider_shared.cpp
45>lib_webrtc.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_webrtc\lib_webrtc.dir\Debug\lib_webrtc.lib
46>lib_lottie.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_lottie\lib_lottie.dir\Debug\lib_lottie.lib
47>------ 已启动全部重新生成: 项目: td_ui, 配置: Debug Win32 ------
47>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
47>cmake_pch.cxx
47>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory
47>已完成生成项目“td_ui.vcxproj”的操作 - 失败。
48>------ 已启动全部重新生成: 项目: Telegram_autogen, 配置: Debug Win32 ------
48>1>Automatic MOC for target Telegram
48>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
49>------ 已启动全部重新生成: 项目: Telegram, 配置: Debug Win32 ------
49>1>Automatic RCC for Resources/qrc/emoji_1.qrc
49>3>Automatic RCC for Resources/qrc/emoji_7.qrc
49>4>Automatic RCC for Resources/qrc/telegram/export.qrc
49>2>Automatic RCC for Resources/qrc/emoji_4.qrc
49>1>Automatic RCC for Resources/qrc/emoji_2.qrc
49>4>Automatic RCC for Resources/qrc/telegram/telegram.qrc
49>2>Automatic RCC for Resources/qrc/emoji_5.qrc
49>3>Automatic RCC for Resources/qrc/emoji_preview.qrc
49>1>Automatic RCC for Resources/qrc/emoji_3.qrc
49>2>Automatic RCC for Resources/qrc/emoji_6.qrc
49>4>Automatic RCC for Resources/qrc/telegram/sounds.qrc
49>3>Automatic RCC for Resources/qrc/telegram/animations.qrc
49>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
49>cmake_pch.cxx
49>C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/SourceFiles/stdafx.h(129,1): fatal  error C1083: 无法打开包括文件: “styles/palette.h”: No such file or directory
49>已完成生成项目“Telegram.vcxproj”的操作 - 失败。
========== “全部重新生成”: 34 成功，15 失败，0已跳过 ==========
========= 重新生成 开始于 9:46，并花费了 02:56.390 分钟 ==========

<pre>
已启动重新生成...
1>------ 已启动全部重新生成: 项目: ZERO_CHECK, 配置: Debug Win32 ------
1>1>Checking Build System
2>------ 已启动全部重新生成: 项目: win_directx_helper, 配置: Debug Win32 ------
3>------ 已启动全部重新生成: 项目: td_scheme_scheme, 配置: Debug Win32 ------
4>------ 已启动全部重新生成: 项目: lib_tgvoip_bundled, 配置: Debug Win32 ------
5>------ 已启动全部重新生成: 项目: lib_tgcalls, 配置: Debug Win32 ------
3>Generating scheme (td_scheme)
2>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/win_directx_helper/CMakeLists.txt
5>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
4>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
2>win_directx_helper.cpp
5>Instance.cpp
4>BlockingQueue.cpp
5>AudioDeviceHelper.cpp
5>ChannelManager.cpp
4>Buffers.cpp
4>CongestionControl.cpp
4>EchoCanceller.cpp
5>CodecSelectHelper.cpp
4>JitterBuffer.cpp
5>CryptoHelper.cpp
4>logging.cpp
5>EncryptedConnection.cpp
4>MediaStreamItf.cpp
3>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
5>FakeAudioDeviceModule.cpp
4>OpusDecoder.cpp
4>OpusEncoder.cpp
4>VoIPController.cpp
5>FieldTrialsConfig.cpp
4>VoIPGroupController.cpp
2>win_directx_helper.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\win_directx_helper\Debug\libwin_directx_helper.a
4>VoIPServerConfig.cpp
5>InstanceImpl.cpp
4>AudioInput.cpp
5>LogSinkImpl.cpp
5>Manager.cpp
5>MediaManager.cpp
4>AudioOutput.cpp
6>------ 已启动全部重新生成: 项目: lib_crl, 配置: Debug Win32 ------
5>Message.cpp
4>Resampler.cpp
4>NetworkSocket.cpp
4>PacketReassembler.cpp
4>MessageThread.cpp
4>AudioIO.cpp
4>ScreamCongestionController.cpp
4>VideoSource.cpp
4>VideoRenderer.cpp
4>json11.cpp
7>------ 已启动全部重新生成: 项目: external_kimageformats_init, 配置: Debug Win32 ------
4>NetworkSocketWinsock.cpp
4>AudioInputWave.cpp
4>AudioOutputWave.cpp
5>NetworkManager.cpp
5>SctpDataChannelProviderInterfaceImpl.cpp
6>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_crl/CMakeLists.txt
4>AudioOutputWASAPI.cpp
5>StaticThreads.cpp
5>TurnCustomizerImpl.cpp
4>AudioInputWASAPI.cpp
4>WindowsSpecific.cpp
5>VideoCaptureInterface.cpp
5>VideoCaptureInterfaceImpl.cpp
5>gzip.cpp
5>ContentNegotiation.cpp
6>crl_common_fp_exceptions.cpp
7>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/kimageformats/CMakeLists.txt
6>crl_common_list.cpp
6>crl_common_on_main.cpp
6>crl_common_queue.cpp
6>crl_dispatch_async.cpp
6>crl_dispatch_queue.cpp
6>crl_dispatch_semaphore.cpp
6>crl_qt_async.cpp
6>crl_qt_semaphore.cpp
6>crl_winapi_async.cpp
6>crl_winapi_fp_exceptions.cpp
6>crl_winapi_list.cpp
6>crl_winapi_semaphore.cpp
6>crl_winapi_time.cpp
4>lib_tgvoip_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\Debug\liblib_tgvoip_bundled.a
5>DirectNetworkingImpl.cpp
5>ExternalSignalingConnection.cpp
7>init.cpp
5>InstanceV2ReferenceImpl.cpp
6>crl_object_on_thread.cpp
5>InstanceV2Impl.cpp
6>crl_time.cpp
8>------ 已启动全部重新生成: 项目: external_kimageformats, 配置: Debug Win32 ------
5>NativeNetworkingImpl.cpp
6>lib_crl.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_crl\lib_crl.dir\Debug\lib_crl.lib
7>external_kimageformats_init.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\kimageformats\external_kimageformats_init.dir\Debug\external_kimageformats_init.lib
9>------ 已启动全部重新生成: 项目: lib_base, 配置: Debug Win32 ------
5>ReflectorPort.cpp
10>------ 已启动全部重新生成: 项目: external_kcoreaddons, 配置: Debug Win32 ------
9>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_base/CMakeLists.txt
10>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/kcoreaddons/CMakeLists.txt
9>cmake_pch.cxx
5>ReflectorRelayPortFactory.cpp
5>Signaling.cpp
5>SignalingConnection.cpp
5>SignalingEncryption.cpp
5>SignalingSctpConnection.cpp
10>kurlmimedata.cpp
10>ksandbox.cpp
5>InstanceV2_4_0_0Impl.cpp
5>Signaling_4_0_0.cpp
5>DesktopCaptureSource.cpp
5>DesktopCaptureSourceHelper.cpp
8>Generating __/__/__/Telegram/ThirdParty/kimageformats/src/imageformats/moc_avif_p.cpp
8>Generating __/__/__/Telegram/ThirdParty/kimageformats/src/imageformats/moc_heif_p.cpp
5>DesktopCaptureSourceManager.cpp
5>AVIOContextImpl.cpp
10>external_kcoreaddons.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\kcoreaddons\Debug\libexternal_kcoreaddons.a
9>base_battery_saving_win.cpp
9>base_file_utilities_win.cpp
9>base_global_shortcuts_win.cpp
9>base_info_win.cpp
8>Generating __/__/__/Telegram/ThirdParty/kimageformats/src/imageformats/moc_jxl_p.cpp
5>AudioStreamingPart.cpp
9>base_last_input_win.cpp
5>AudioStreamingPartInternal.cpp
11>------ 已启动全部重新生成: 项目: codegen_common, 配置: Debug Win32 ------
9>base_layout_switch_win.cpp
9>base_network_reachability_win.cpp
5>AudioStreamingPartPersistentDecoder.cpp
9>base_power_save_blocker_win.cpp
9>base_process_win.cpp
9>base_system_media_controls_win.cpp
5>GroupInstanceCustomImpl.cpp
9>base_url_scheme_win.cpp
9>base_windows_safe_library.cpp
5>GroupJoinPayloadInternal.cpp
9>base_windows_winrt.cpp
5>GroupNetworkManager.cpp
5>StreamingMediaContext.cpp
8>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/kimageformats/CMakeLists.txt
9>base_windows_wrl.cpp
9>base_platform_info.cpp
9>algorithm.cpp
9>assertion.cpp
5>VideoStreamingPart.cpp
9>base_file_utilities.cpp
9>battery_saving.cpp
9>bytes.cpp
11>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/common/CMakeLists.txt
9>call_delayed.cpp
9>crc32hash.cpp
9>concurrent_timer.cpp
9>debug_log.cpp
9>event_filter.cpp
9>file_lock_win.cpp
9>global_shortcuts_generic.cpp
9>integration.cpp
9>last_user_input.cpp
9>network_reachability.cpp
9>options.cpp
9>parse_helper.cpp
9>power_save_blocker.cpp
9>qthelp_url.cpp
9>random.cpp
9>runtime_composer.cpp
9>single_instance.cpp
9>timer.cpp
9>unixtime.cpp
5>DesktopInterface.cpp
9>qt_tab_key.cpp
5>VideoCapturerInterfaceImpl.cpp
8>avif.cpp
8>heif.cpp
8>jxl.cpp
8>moc_avif_p.cpp
5>VideoCapturerTrackSource.cpp
11>basic_tokenized_file.cpp
11>checked_utf8_string.cpp
11>clean_file.cpp
11>cpp_file.cpp
11>logging.cpp
9>lib_base.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_base\Debug\liblib_base.a
8>moc_heif_p.cpp
8>moc_jxl_p.cpp
5>VideoCameraCapturer.cpp
5>json11.cpp
12>------ 已启动全部重新生成: 项目: lib_tl, 配置: Debug Win32 ------
11>codegen_common.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\codegen\codegen\common\codegen_common.dir\Debug\codegen_common.lib
13>------ 已启动全部重新生成: 项目: codegen_style, 配置: Debug Win32 ------
13>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/style/CMakeLists.txt
13>generator.cpp
13>main.cpp
13>module.cpp
13>options.cpp
13>parsed_file.cpp
12>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_tl/CMakeLists.txt
13>processor.cpp
13>structure_types.cpp
12>tl_basic_types.cpp
5>lib_tgcalls.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\Debug\liblib_tgcalls.a
14>------ 已启动全部重新生成: 项目: lib_tgcalls_legacy, 配置: Debug Win32 ------
8>external_kimageformats.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\kimageformats\Debug\libexternal_kimageformats.a
15>------ 已启动全部重新生成: 项目: external_qt_static_plugins, 配置: Debug Win32 ------
12>lib_tl.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_tl\lib_tl.dir\Debug\lib_tl.lib
16>------ 已启动全部重新生成: 项目: td_scheme, 配置: Debug Win32 ------
15>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/qt/qt_static_plugins/CMakeLists.txt
14>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
15>qt_static_plugins.cpp
16>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
13>LINK : fatal error LNK1104: 无法打开文件“dxguid.obj”
13>已完成生成项目“codegen_style.vcxproj”的操作 - 失败。
17>------ 已启动全部重新生成: 项目: td_ui_styles, 配置: Debug Win32 ------
16>scheme.cpp
16>scheme-dump_to_text.cpp
17>Generating styles (td_ui)
17>'codegen\codegen\style\Debug\codegen_style.exe' 不是内部或外部命令，也不是可运行的程序
17>或批处理文件。
17>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\5d71192b9cd298b8cb10a9827e4946ce\td_ui_style.timestamp.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\868136f9c1deeb6abe216f3361c29fd3\td_ui_styles.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\CMakeLists.txt”的自定义生成已退出，代码为 9009。
17>已完成生成项目“td_ui_styles.vcxproj”的操作 - 失败。
18>------ 已启动全部重新生成: 项目: lib_ui_styles, 配置: Debug Win32 ------
18>Generating styles (lib_ui)
18>'..\codegen\codegen\style\Debug\codegen_style.exe' 不是内部或外部命令，也不是可运行的程序
18>或批处理文件。
18>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\14900836e4a4ec0e1ba31f97b7dd2f36\lib_ui_style.timestamp.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3b7f912706227bff2b098755786bcfe6\lib_ui_styles.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\CMakeLists.txt”的自定义生成已退出，代码为 9009。
18>已完成生成项目“lib_ui_styles.vcxproj”的操作 - 失败。
19>------ 已启动全部重新生成: 项目: lib_ui_palette, 配置: Debug Win32 ------
19>Generating palette (lib_ui)
19>'..\codegen\codegen\style\Debug\codegen_style.exe' 不是内部或外部命令，也不是可运行的程序
19>或批处理文件。
19>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\colors.palette;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3b7f912706227bff2b098755786bcfe6\lib_ui_palette.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\CMakeLists.txt”的自定义生成已退出，代码为 9009。
19>已完成生成项目“lib_ui_palette.vcxproj”的操作 - 失败。
20>------ 已启动全部重新生成: 项目: codegen_lang, 配置: Debug Win32 ------
20>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/lang/CMakeLists.txt
20>generator.cpp
20>main.cpp
20>options.cpp
20>parsed_file.cpp
14>InstanceImplLegacy.cpp
15>external_qt_static_plugins.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\qt\qt_static_plugins\external_qt_static_plugins.dir\Debug\external_qt_static_plugins.lib
20>processor.cpp
21>------ 已启动全部重新生成: 项目: codegen_emoji, 配置: Debug Win32 ------
14>lib_tgcalls_legacy.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\Debug\liblib_tgcalls_legacy.a
21>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/emoji/CMakeLists.txt
22>------ 已启动全部重新生成: 项目: lib_stripe, 配置: Debug Win32 ------
21>data.cpp
21>data_old.cpp
21>data_read.cpp
21>generator.cpp
22>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
20>LINK : fatal error LNK1104: 无法打开文件“dxguid.obj”
21>main.cpp
20>已完成生成项目“codegen_lang.vcxproj”的操作 - 失败。
23>------ 已启动全部重新生成: 项目: td_lang_lang, 配置: Debug Win32 ------
21>options.cpp
22>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): warning MSB8065: 项“C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\CMakeLists.txt”的自定义生成成功，但尚未创建指定的输出“c:\users\admin\desktop\tbuild\buildpath\tdesktop\out\telegram\cmakefiles\generate.stamp”。这可能会导致增量生成无法正常工作。
21>replaces.cpp
22>cmake_pch.cxx
23>Generating lang (td_lang)
23>'codegen\codegen\lang\Debug\codegen_lang.exe' 不是内部或外部命令，也不是可运行的程序
23>或批处理文件。
23>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\a8ad2d315d3f78d762dc15d12a0bb9e3\lang_auto.timestamp.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\868136f9c1deeb6abe216f3361c29fd3\td_lang_lang.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\CMakeLists.txt”的自定义生成已退出，代码为 9009。
23>已完成生成项目“td_lang_lang.vcxproj”的操作 - 失败。
24>------ 已启动全部重新生成: 项目: lib_storage, 配置: Debug Win32 ------
24>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_storage/CMakeLists.txt
24>cmake_pch.cxx
16>td_scheme.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\td_scheme.dir\Debug\td_scheme.lib
21>LINK : fatal error LNK1104: 无法打开文件“dxguid.obj”
22>stripe_api_client.cpp
22>stripe_card.cpp
22>stripe_card_params.cpp
22>stripe_card_validator.cpp
21>已完成生成项目“codegen_emoji.vcxproj”的操作 - 失败。
25>------ 已启动全部重新生成: 项目: lib_ui_emoji, 配置: Debug Win32 ------
22>stripe_decode.cpp
22>stripe_error.cpp
22>stripe_form_encoder.cpp
22>stripe_token.cpp
22>smartglocal_api_client.cpp
22>smartglocal_card.cpp
22>smartglocal_error.cpp
22>smartglocal_token.cpp
25>Generating emoji (lib_ui)
25>'..\codegen\codegen\emoji\Debug\codegen_emoji.exe' 不是内部或外部命令，也不是可运行的程序
25>或批处理文件。
25>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3c095a66caa0e85d9452707996a8760a\emoji.timestamp.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3b7f912706227bff2b098755786bcfe6\lib_ui_emoji.rule;C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\CMakeLists.txt”的自定义生成已退出，代码为 9009。
25>已完成生成项目“lib_ui_emoji.vcxproj”的操作 - 失败。
26>------ 已启动全部重新生成: 项目: lib_ui_autogen, 配置: Debug Win32 ------
22>lib_stripe.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_stripe.dir\Debug\lib_stripe.lib
22>已完成生成项目“lib_stripe.vcxproj”的操作。
27>------ 已启动全部重新生成: 项目: external_rlottie_bundled, 配置: Debug Win32 ------
27>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/rlottie/CMakeLists.txt
27>lottieanimation.cpp
27>lottieitem.cpp
27>lottiekeypath.cpp
27>lottieloader.cpp
27>lottiemodel.cpp
27>lottieparser.cpp
27>lottieproxymodel.cpp
27>v_ft_math.cpp
27>v_ft_raster.cpp
27>v_ft_stroker.cpp
27>vregion.cpp
27>vbezier.cpp
27>vbitmap.cpp
28>------ 已启动全部重新生成: 项目: td_mtproto, 配置: Debug Win32 ------
24>storage_clear_legacy.cpp
27>vbrush.cpp
24>storage_clear_legacy_win.cpp
24>storage_databases.cpp
27>vcompositionfunctions.cpp
24>storage_encrypted_file.cpp
27>vdasher.cpp
24>storage_encryption.cpp
24>storage_cache_binlog_reader.cpp
24>storage_cache_cleaner.cpp
24>storage_cache_compactor.cpp
27>vdebug.cpp
24>storage_cache_database.cpp
27>vdrawable.cpp
27>vdrawhelper.cpp
27>vdrawhelper_neon.cpp
24>storage_cache_database_object.cpp
27>vdrawhelper_sse2.cpp
27>velapsedtimer.cpp
24>storage_cache_types.cpp
27>vimageloader.cpp
27>vinterpolator.cpp
27>vmatrix.cpp
27>vpainter.cpp
27>vpath.cpp
27>vpathmesure.cpp
27>vraster.cpp
26>1>Automatic MOC for target lib_ui
28>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
27>vrect.cpp
27>vrle.cpp
26>
24>lib_storage.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_storage\lib_storage.dir\Debug\lib_storage.lib
26>AutoGen error
26>-------------
26>Info error in info file
26>"C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/out/Telegram/lib_ui/CMakeFiles/lib_ui_autogen.dir/AutogenInfo.json":
26>The header file "SRC:/out/Telegram/lib_ui/gen/emoji.h" does not exist.
26>
29>------ 已启动全部重新生成: 项目: td_export, 配置: Debug Win32 ------
26>系统找不到指定的批处理标签 - VCEnd
28>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): warning MSB8065: 项“C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\CMakeLists.txt”的自定义生成成功，但尚未创建指定的输出“c:\users\admin\desktop\tbuild\buildpath\tdesktop\out\telegram\cmakefiles\generate.stamp”。这可能会导致增量生成无法正常工作。
26>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(232,5): error MSB8066: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3b7f912706227bff2b098755786bcfe6\lib_ui_autogen.rule”的自定义生成已退出，代码为 1。
26>已完成生成项目“lib_ui_autogen.vcxproj”的操作 - 失败。
30>------ 已启动全部重新生成: 项目: external_qr_code_generator_bundled, 配置: Debug Win32 ------
28>cmake_pch.cxx
30>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/qr_code_generator/CMakeLists.txt
29>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
30>qrcodegen.cpp
29>cmake_pch.cxx
27>external_rlottie_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\rlottie\Debug\libexternal_rlottie_bundled.a
31>------ 已启动全部重新生成: 项目: external_minizip_bundled, 配置: Debug Win32 ------
31>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/minizip/CMakeLists.txt
31>ioapi.c
31>zip.c
31>unzip.c
30>external_qr_code_generator_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\qr_code_generator\Debug\libexternal_qr_code_generator_bundled.a
32>------ 已启动全部重新生成: 项目: lib_qr, 配置: Debug Win32 ------
31>external_minizip_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\minizip\Debug\libexternal_minizip_bundled.a
33>------ 已启动全部重新生成: 项目: external_lz4_bundled, 配置: Debug Win32 ------
32>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_qr/CMakeLists.txt
32>qr_generate.cpp
33>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/lz4/CMakeLists.txt
33>lz4.c
33>lz4frame.c
33>lz4hc.c
33>xxhash.c
32>lib_qr.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_qr\lib_qr.dir\Debug\lib_qr.lib
33>external_lz4_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\lz4\Debug\libexternal_lz4_bundled.a
34>------ 已启动全部重新生成: 项目: external_hunspell_bundled, 配置: Debug Win32 ------
35>------ 已启动全部重新生成: 项目: lib_ui, 配置: Debug Win32 ------
34>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/hunspell/CMakeLists.txt
34>affentry.cxx
34>affixmgr.cxx
34>csutil.cxx
34>filemgr.cxx
34>hashmgr.cxx
34>hunspell.cxx
34>hunzip.cxx
34>phonet.cxx
34>replist.cxx
34>suggestmgr.cxx
29>export_api_wrap.cpp
28>mtproto_abstract_socket.cpp
29>export_controller.cpp
29>export_settings.cpp
29>export_data_types.cpp
28>mtproto_bound_key_creator.cpp
28>mtproto_dc_key_binder.cpp
28>mtproto_dc_key_creator.cpp
29>export_output_abstract.cpp
28>mtproto_dcenter.cpp
28>mtproto_domain_resolver.cpp
28>mtproto_dump_to_text.cpp
28>mtproto_received_ids_manager.cpp
28>mtproto_rsa_public_key.cpp
28>mtproto_serialized_request.cpp
28>mtproto_tcp_socket.cpp
28>mtproto_tls_socket.cpp
28>mtproto_auth_key.cpp
29>export_output_file.cpp
28>mtproto_concurrent_sender.cpp
34>external_hunspell_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\hunspell\Debug\libexternal_hunspell_bundled.a
28>mtproto_config.cpp
28>mtproto_dc_options.cpp
29>export_output_html.cpp
28>mtproto_dh_utils.cpp
28>mtproto_proxy_data.cpp
36>------ 已启动全部重新生成: 项目: external_cld3_bundled_task_spec.proto, 配置: Debug Win32 ------
28>mtproto_response.cpp
29>export_output_json.cpp
35>1>Automatic RCC for fonts/fonts.qrc
36>Generating protobuf task_spec.proto (external_cld3_bundled)
29>export_output_stats.cpp
35>2>Automatic RCC for qt_conf/win.qrc
28>td_mtproto.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\td_mtproto.dir\Debug\td_mtproto.lib
36>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
35>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_ui/CMakeLists.txt
37>------ 已启动全部重新生成: 项目: external_cld3_bundled_sentence.proto, 配置: Debug Win32 ------
28>已完成生成项目“td_mtproto.vcxproj”的操作。
37>Generating protobuf sentence.proto (external_cld3_bundled)
38>------ 已启动全部重新生成: 项目: external_cld3_bundled_feature_extractor.proto, 配置: Debug Win32 ------
35>cmake_pch.cxx
38>Generating protobuf feature_extractor.proto (external_cld3_bundled)
37>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
38>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
39>------ 已启动全部重新生成: 项目: Telegram_midl, 配置: Debug Win32 ------
39>Generating headers from IDLs (Telegram)
40>------ 已启动全部重新生成: 项目: external_cld3_bundled, 配置: Debug Win32 ------
39>Microsoft (R) 32b/64b MIDL Compiler Version 8.01.0626
39>Copyright (c) Microsoft Corporation. All rights reserved.
39>Processing C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\platform\win\windows_quiethours.idl
39>windows_quiethours.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\oaidl.idl
39>oaidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\objidl.idl
39>objidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\unknwn.idl
39>unknwn.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\wtypes.idl
39>wtypes.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\wtypesbase.idl
39>wtypesbase.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\basetsd.h
39>basetsd.h
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\guiddef.h
39>guiddef.h
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\oaidl.acf
39>oaidl.acf
39>Microsoft (R) 32b/64b MIDL Compiler Version 8.01.0626
39>Copyright (c) Microsoft Corporation. All rights reserved.
39>Processing C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\platform\win\windows_toastactivator.idl
39>windows_toastactivator.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\oaidl.idl
39>oaidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\objidl.idl
39>objidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\unknwn.idl
39>unknwn.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\wtypes.idl
39>wtypes.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\wtypesbase.idl
39>wtypesbase.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\basetsd.h
39>basetsd.h
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\shared\guiddef.h
39>guiddef.h
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\ocidl.idl
39>ocidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\oleidl.idl
39>oleidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\servprov.idl
39>servprov.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\urlmon.idl
39>urlmon.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\msxml.idl
39>msxml.idl
29>td_export.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\td_export.dir\Debug\td_export.lib
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\oaidl.acf
39>oaidl.acf
40>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22000.0\um\ocidl.acf
39>ocidl.acf
39>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
40>feature_extractor.pb.cc
40>sentence.pb.cc
40>task_spec.pb.cc
40>base.cc
40>embedding_feature_extractor.cc
40>embedding_network.cc
40>feature_extractor.cc
40>feature_types.cc
40>fml_parser.cc
40>language_identifier_features.cc
35>mocs_compilation_Debug.cpp
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\lib_ui_autogen\mocs_compilation_Debug.cpp”: No such file or directory
35>palette.cpp
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\palette.cpp”: No such file or directory
35>style_basic.cpp
40>lang_id_nn_params.cc
40>nnet_language_identifier.cc
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\style_basic.cpp”: No such file or directory
35>style_layers.cpp
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\style_layers.cpp”: No such file or directory
35>style_widgets.cpp
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\style_widgets.cpp”: No such file or directory
35>emoji.cpp
35>emoji_suggestions_data.cpp
35>dpr_icon.cpp
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\emoji.cpp”: No such file or directory
35>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\emoji_suggestions_data.cpp”: No such file or directory
35>animation_value.cpp
35>animations.cpp
35>cross_animation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/cross_animation.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\cross_animation.cpp)
35>cross_line.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/cross_line.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\cross_line.cpp)
35>fade_animation.cpp
35>frame_generator.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\animation_value.cpp)
35>gradient.cpp
40>registry.cc
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/fade_animation.h(11,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\fade_animation.cpp)
35>numbers_animation.cpp
40>relevant_script_feature.cc
35>panel_animation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/panel_animation.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\panel_animation.cpp)
35>path_shift_gradient.cpp
35>radial_animation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\numbers_animation.cpp)
35>ripple_animation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/ripple_animation.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\ripple_animation.cpp)
40>sentence_features.cc
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\radial_animation.cpp)
35>round_area_with_shadow.cpp
35>show_animation.cpp
35>slide_animation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\round_area_with_shadow.cpp)
35>spoiler_mess.cpp
35>gl_detection.cpp
35>gl_image.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\spoiler_mess.cpp)
35>gl_math.cpp
40>task_context.cc
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\show_animation.cpp(11,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory
35>gl_primitives.cpp
35>gl_shader.cpp
40>task_context_params.cc
40>unicodetext.cc
35>gl_surface.cpp
40>utils.cc
35>gl_window.cpp
35>image_prepare.cpp
40>workspace.cc
35>box_content.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\image\image_prepare.cpp)
35>box_layer_widget.cpp
40>generated_entities.cc
40>getonescriptspan.cc
40>utf8statetable.cc
40>offsetmap.cc
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\box_content.cpp)
35>generic_box.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\gl\gl_surface.cpp)
35>layer_manager.cpp
40>text_processing.cc
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\box_layer_widget.cpp)
40>fixunicodevalue.cc
35>layer_widget.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\generic_box.cpp)
35>show.cpp
35>arcs.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\layer_widget.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/paint/arcs.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\arcs.cpp)
35>blob.cpp
35>blobs.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\blob.cpp)
35>blobs_linear.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\blobs.cpp)
35>ui_window_shadow_win.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\blobs_linear.cpp)
35>ui_window_title_win.cpp
40>external_cld3_bundled.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\cld3\Debug\libexternal_cld3_bundled.a
35>ui_window_win.cpp
35>ui_windows_direct_manipulation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_window_shadow_win.cpp(12,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory
35>ui_utility_win.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_window_title_win.cpp)
35>ui_platform_window_title.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_window_win.cpp(19,1): fatal  error C1083: 无法打开包括文件: “styles/palette.h”: No such file or directory
35>ui_platform_window.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\ui_platform_window_title.cpp)
35>style_core.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/popup_menu.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_utility_win.cpp)
35>style_core_color.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/style/style_core_palette.h(10,1): fatal  error C1083: 无法打开包括文件: “styles/palette.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core_color.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core.cpp)
35>style_core_custom_font.cpp
35>style_core_direction.cpp
35>style_core_font.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\ui_platform_window.cpp)
35>style_core_icon.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/style/style_core_palette.h(10,1): fatal  error C1083: 无法打开包括文件: “styles/palette.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core_icon.cpp)
35>style_core_palette.cpp
35>style_core_scale.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/style/style_core_palette.h(10,1): fatal  error C1083: 无法打开包括文件: “styles/palette.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core_palette.cpp)
35>style_core_types.cpp
35>style_palette_colorizer.cpp
35>custom_emoji_instance.cpp
35>text.cpp
35>text_block.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_block.cpp)
35>text_custom_emoji.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text.cpp)
35>text_entity.cpp
35>text_parser.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_parser.cpp)
35>text_renderer.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\custom_emoji_instance.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_entity.cpp)
35>text_spoiler_data.cpp
35>text_utilities.cpp
35>text_variant.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_renderer.cpp)
35>toast.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_spoiler_data.cpp)
35>toast_manager.cpp
35>toast_widget.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\toast\toast.cpp)
35>box_content_divider.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\toast\toast_widget.cpp)
35>buttons.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\toast\toast_manager.cpp)
35>call_button.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\box_content_divider.cpp(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_layers.h”: No such file or directory
35>checkbox.cpp
35>dropdown_menu.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/dropdown_menu.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\dropdown_menu.cpp)
35>icon_button_with_text.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\buttons.cpp)
35>inner_dropdown.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/inner_dropdown.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\inner_dropdown.cpp)
35>input_fields.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\input_fields.cpp)
35>labels.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\call_button.cpp)
35>time_part_input.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\fields\time_part_input.cpp)
35>menu.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\checkbox.cpp)
35>menu_action.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\icon_button_with_text.cpp)
35>menu_add_action_callback.cpp
35>menu_add_action_callback_factory.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_action.cpp)
35>menu_common.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_add_action_callback_factory.cpp)
35>menu_item_base.cpp
35>menu_multiline_action.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\labels.cpp)
35>menu_separator.cpp
35>menu_toggle.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_toggle.cpp)
35>elastic_scroll.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_multiline_action.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_item_base.cpp)
35>popup_menu.cpp
35>rp_window.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/popup_menu.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\popup_menu.cpp)
35>separate_panel.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_separator.cpp)
35>scroll_area.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\separate_panel.cpp)
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/scroll_area.h(13,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\elastic_scroll.cpp)
35>side_bar_button.cpp
35>shadow.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/scroll_area.h(13,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\scroll_area.cpp)
35>time_input.cpp
35>tooltip.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\shadow.cpp(10,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory
35>fade_wrap.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\side_bar_button.cpp)
35>follow_slide_wrap.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\tooltip.cpp)
35>padding_wrap.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/fade_animation.h(11,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\fade_wrap.cpp)
35>slide_wrap.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\time_input.cpp)
35>vertical_layout.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\follow_slide_wrap.cpp(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory
35>vertical_layout_reorder.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\slide_wrap.cpp(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_basic.h”: No such file or directory
35>abstract_button.cpp
35>animated_icon.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/scroll_area.h(13,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\vertical_layout_reorder.cpp)
35>basic_click_handlers.cpp
35>cached_special_layer_shadow_corners.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/ripple_animation.h(9,1): fatal  error C1083: 无法打开包括文件: “styles/style_widgets.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\cached_special_layer_shadow_corners.cpp)
35>click_handler.cpp
35>delayed_activation.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\basic_click_handlers.cpp)
35>dragging_scroll_manager.cpp
35>emoji_config.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\emoji_config.cpp)
35>inactive_press.cpp
35>integration.cpp
35>main_queue_processor.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\integration.cpp)
35>ph.cpp
35>round_rect.cpp
35>rp_widget.cpp
35>ui_utility.cpp
35>emoji_suggestions.cpp
35>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\emoji_suggestions\emoji_suggestions.cpp(10,1): fatal  error C1083: 无法打开包括文件: “emoji_suggestions_data.h”: No such file or directory
35>qrc_win.cpp
35>qrc_fonts.cpp
35>已完成生成项目“lib_ui.vcxproj”的操作 - 失败。
41>------ 已启动全部重新生成: 项目: td_lang, 配置: Debug Win32 ------
42>------ 已启动全部重新生成: 项目: lib_webview, 配置: Debug Win32 ------
43>------ 已启动全部重新生成: 项目: lib_spellcheck, 配置: Debug Win32 ------
44>------ 已启动全部重新生成: 项目: lib_ffmpeg, 配置: Debug Win32 ------
43>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_spellcheck/CMakeLists.txt
42>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_webview/CMakeLists.txt
44>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
41>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
42>webview_dialog.cpp
43>cmake_pch.cxx
44>ffmpeg_frame_generator.cpp
42>webview_embed.cpp
42>webview_win.cpp
44>ffmpeg_utility.cpp
42>webview_windows_edge_chromium.cpp
42>webview_windows_edge_html.cpp
41>cmake_pch.cxx
42>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_webview\webview\webview_dialog.cpp)
44>lib_ffmpeg.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ffmpeg.dir\Debug\lib_ffmpeg.lib
43>spellcheck_win.cpp
43>language_cld3.cpp
43>hunspell_controller.cpp
43>spellcheck_utils.cpp
41>lang_auto.cpp
41>c1xx : fatal  error C1083: 无法打开源文件: “C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\gen\lang_auto.cpp”: No such file or directory
41>lang_file_parser.cpp
41>lang_keys.cpp
41>lang_tag.cpp
41>lang_text_entity.cpp
41>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang/lang_keys.h(10,1): fatal  error C1083: 无法打开包括文件: “lang_auto.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang\lang_keys.cpp)
41>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang/lang_keys.h(10,1): fatal  error C1083: 无法打开包括文件: “lang_auto.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang\lang_file_parser.cpp)
41>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang/lang_keys.h(10,1): fatal  error C1083: 无法打开包括文件: “lang_auto.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang\lang_tag.cpp)
43>spelling_highlighter.cpp
43>spelling_highlighter_helper.cpp
43>spellcheck_value.cpp
43>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_spellcheck\spellcheck\spelling_highlighter.cpp)
41>已完成生成项目“td_lang.vcxproj”的操作 - 失败。
43>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory (编译源文件 C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_spellcheck\spellcheck\spelling_highlighter_helper.cpp)
42>已完成生成项目“lib_webview.vcxproj”的操作 - 失败。
45>------ 已启动全部重新生成: 项目: lib_webrtc, 配置: Debug Win32 ------
46>------ 已启动全部重新生成: 项目: lib_lottie, 配置: Debug Win32 ------
46>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_lottie/CMakeLists.txt
45>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_webrtc/CMakeLists.txt
45>cmake_pch.cxx
46>lottie_frame_provider_direct.cpp
46>lottie_frame_renderer.cpp
46>lottie_animation.cpp
46>lottie_common.cpp
46>lottie_frame_generator.cpp
43>已完成生成项目“lib_spellcheck.vcxproj”的操作 - 失败。
46>lottie_multi_player.cpp
46>lottie_single_player.cpp
46>lottie_icon.cpp
45>webrtc_audio_input_tester.cpp
45>webrtc_create_adm.cpp
45>webrtc_media_devices.cpp
45>webrtc_video_track.cpp
45>webrtc_openal_adm.cpp
46>lottie_cache.cpp
45>webrtc_loopback_adm_win.cpp
46>lottie_cache_frame_storage.cpp
46>lottie_frame_provider_cached.cpp
46>lottie_frame_provider_cached_multi.cpp
46>lottie_frame_provider_shared.cpp
45>lib_webrtc.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_webrtc\lib_webrtc.dir\Debug\lib_webrtc.lib
46>lib_lottie.vcxproj -> C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_lottie\lib_lottie.dir\Debug\lib_lottie.lib
47>------ 已启动全部重新生成: 项目: td_ui, 配置: Debug Win32 ------
47>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
47>cmake_pch.cxx
47>C:\Users\admin\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: 无法打开包括文件: “emoji.h”: No such file or directory
47>已完成生成项目“td_ui.vcxproj”的操作 - 失败。
48>------ 已启动全部重新生成: 项目: Telegram_autogen, 配置: Debug Win32 ------
48>1>Automatic MOC for target Telegram
48>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
49>------ 已启动全部重新生成: 项目: Telegram, 配置: Debug Win32 ------
49>1>Automatic RCC for Resources/qrc/emoji_1.qrc
49>3>Automatic RCC for Resources/qrc/emoji_7.qrc
49>4>Automatic RCC for Resources/qrc/telegram/export.qrc
49>2>Automatic RCC for Resources/qrc/emoji_4.qrc
49>1>Automatic RCC for Resources/qrc/emoji_2.qrc
49>4>Automatic RCC for Resources/qrc/telegram/telegram.qrc
49>2>Automatic RCC for Resources/qrc/emoji_5.qrc
49>3>Automatic RCC for Resources/qrc/emoji_preview.qrc
49>1>Automatic RCC for Resources/qrc/emoji_3.qrc
49>2>Automatic RCC for Resources/qrc/emoji_6.qrc
49>4>Automatic RCC for Resources/qrc/telegram/sounds.qrc
49>3>Automatic RCC for Resources/qrc/telegram/animations.qrc
49>Building Custom Rule C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
49>cmake_pch.cxx
49>C:/Users/admin/Desktop/TBuild/BuildPath/tdesktop/Telegram/SourceFiles/stdafx.h(129,1): fatal  error C1083: 无法打开包括文件: “styles/palette.h”: No such file or directory
49>已完成生成项目“Telegram.vcxproj”的操作 - 失败。
========== “全部重新生成”: 34 成功，15 失败，0已跳过 ==========
========= 重新生成 开始于 9:46，并花费了 02:56.390 分钟 ==========

<pre>

<pre>
Build started...
1>------ Build started: Project: ZERO_CHECK, Configuration: Debug Win32 ------
1>1>Checking Build System
2>------ Build started: Project: win_directx_helper, Configuration: Debug Win32 ------
3>------ Build started: Project: td_scheme_scheme, Configuration: Debug Win32 ------
4>------ Build started: Project: lib_tgvoip_bundled, Configuration: Debug Win32 ------
5>------ Build started: Project: lib_tgcalls, Configuration: Debug Win32 ------
4>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
4>BlockingQueue.cpp
4>Buffers.cpp
4>CongestionControl.cpp
4>EchoCanceller.cpp
3>Generating scheme (td_scheme)
2>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/win_directx_helper/CMakeLists.txt
2>win_directx_helper.cpp
4>JitterBuffer.cpp
4>logging.cpp
3>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
4>MediaStreamItf.cpp
4>OpusDecoder.cpp
5>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
2>win_directx_helper.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\cmake\win_directx_helper\Debug\libwin_directx_helper.a
4>OpusEncoder.cpp
4>VoIPController.cpp
4>VoIPGroupController.cpp
6>------ Build started: Project: lib_crl, Configuration: Debug Win32 ------
4>VoIPServerConfig.cpp
7>------ Build started: Project: external_kimageformats_init, Configuration: Debug Win32 ------
4>AudioInput.cpp
4>AudioOutput.cpp
4>Resampler.cpp
4>NetworkSocket.cpp
5>Instance.cpp
7>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/kimageformats/CMakeLists.txt
5>AudioDeviceHelper.cpp
5>ChannelManager.cpp
5>CodecSelectHelper.cpp
4>PacketReassembler.cpp
4>MessageThread.cpp
4>AudioIO.cpp
5>CryptoHelper.cpp
4>ScreamCongestionController.cpp
4>VideoSource.cpp
5>EncryptedConnection.cpp
4>VideoRenderer.cpp
4>json11.cpp
5>FakeAudioDeviceModule.cpp
4>NetworkSocketWinsock.cpp
4>AudioInputWave.cpp
4>AudioOutputWave.cpp
4>AudioOutputWASAPI.cpp
5>FieldTrialsConfig.cpp
5>InstanceImpl.cpp
6>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_crl/CMakeLists.txt
5>LogSinkImpl.cpp
4>AudioInputWASAPI.cpp
4>WindowsSpecific.cpp
7>init.cpp
5>Manager.cpp
5>MediaManager.cpp
5>Message.cpp
6>crl_common_fp_exceptions.cpp
6>crl_common_list.cpp
6>crl_common_on_main.cpp
6>crl_common_queue.cpp
6>crl_dispatch_async.cpp
6>crl_dispatch_queue.cpp
6>crl_dispatch_semaphore.cpp
6>crl_qt_async.cpp
6>crl_qt_semaphore.cpp
6>crl_winapi_async.cpp
6>crl_winapi_fp_exceptions.cpp
6>crl_winapi_list.cpp
6>crl_winapi_semaphore.cpp
6>crl_winapi_time.cpp
4>lib_tgvoip_bundled.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\Debug\liblib_tgvoip_bundled.a
5>NetworkManager.cpp
5>SctpDataChannelProviderInterfaceImpl.cpp
6>crl_object_on_thread.cpp
6>crl_time.cpp
8>------ Build started: Project: external_kimageformats, Configuration: Debug Win32 ------
7>external_kimageformats_init.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\kimageformats\external_kimageformats_init.dir\Debug\external_kimageformats_init.lib
5>StaticThreads.cpp
5>TurnCustomizerImpl.cpp
6>lib_crl.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_crl\lib_crl.dir\Debug\lib_crl.lib
8>Generating __/__/__/Telegram/ThirdParty/kimageformats/src/imageformats/moc_avif_p.cpp
9>------ Build started: Project: external_kcoreaddons, Configuration: Debug Win32 ------
5>VideoCaptureInterface.cpp
5>VideoCaptureInterfaceImpl.cpp
9>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/kcoreaddons/CMakeLists.txt
5>gzip.cpp
10>------ Build started: Project: lib_base, Configuration: Debug Win32 ------
5>ContentNegotiation.cpp
5>DirectNetworkingImpl.cpp
8>Generating __/__/__/Telegram/ThirdParty/kimageformats/src/imageformats/moc_heif_p.cpp
5>ExternalSignalingConnection.cpp
10>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_base/CMakeLists.txt
5>InstanceV2ReferenceImpl.cpp
9>kurlmimedata.cpp
9>ksandbox.cpp
5>InstanceV2Impl.cpp
8>Generating __/__/__/Telegram/ThirdParty/kimageformats/src/imageformats/moc_jxl_p.cpp
10>cmake_pch.cxx
8>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/kimageformats/CMakeLists.txt
5>NativeNetworkingImpl.cpp
9>external_kcoreaddons.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\kcoreaddons\Debug\libexternal_kcoreaddons.a
11>------ Build started: Project: codegen_common, Configuration: Debug Win32 ------
5>ReflectorPort.cpp
11>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/common/CMakeLists.txt
8>avif.cpp
8>heif.cpp
8>jxl.cpp
8>moc_avif_p.cpp
5>ReflectorRelayPortFactory.cpp
11>basic_tokenized_file.cpp
11>checked_utf8_string.cpp
11>clean_file.cpp
11>cpp_file.cpp
8>moc_heif_p.cpp
8>moc_jxl_p.cpp
11>logging.cpp
5>Signaling.cpp
5>SignalingConnection.cpp
5>SignalingEncryption.cpp
5>SignalingSctpConnection.cpp
10>base_battery_saving_win.cpp
10>base_file_utilities_win.cpp
10>base_global_shortcuts_win.cpp
10>base_info_win.cpp
5>InstanceV2_4_0_0Impl.cpp
5>Signaling_4_0_0.cpp
10>base_last_input_win.cpp
10>base_layout_switch_win.cpp
10>base_network_reachability_win.cpp
10>base_power_save_blocker_win.cpp
5>DesktopCaptureSource.cpp
11>codegen_common.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\codegen\codegen\common\codegen_common.dir\Debug\codegen_common.lib
8>external_kimageformats.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\kimageformats\Debug\libexternal_kimageformats.a
10>base_process_win.cpp
10>base_system_media_controls_win.cpp
10>base_url_scheme_win.cpp
10>base_windows_safe_library.cpp
12>------ Build started: Project: codegen_lang, Configuration: Debug Win32 ------
10>base_windows_winrt.cpp
10>base_windows_wrl.cpp
10>base_platform_info.cpp
10>algorithm.cpp
10>assertion.cpp
10>base_file_utilities.cpp
10>battery_saving.cpp
10>bytes.cpp
10>call_delayed.cpp
10>crc32hash.cpp
10>concurrent_timer.cpp
10>debug_log.cpp
10>event_filter.cpp
10>file_lock_win.cpp
10>global_shortcuts_generic.cpp
10>integration.cpp
5>DesktopCaptureSourceHelper.cpp
10>last_user_input.cpp
5>DesktopCaptureSourceManager.cpp
5>AVIOContextImpl.cpp
10>network_reachability.cpp
5>AudioStreamingPart.cpp
10>options.cpp
10>parse_helper.cpp
13>------ Build started: Project: external_qt_static_plugins, Configuration: Debug Win32 ------
5>AudioStreamingPartInternal.cpp
10>power_save_blocker.cpp
10>qthelp_url.cpp
5>AudioStreamingPartPersistentDecoder.cpp
10>random.cpp
10>runtime_composer.cpp
10>single_instance.cpp
10>timer.cpp
10>unixtime.cpp
10>qt_tab_key.cpp
5>GroupInstanceCustomImpl.cpp
12>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/lang/CMakeLists.txt
5>GroupJoinPayloadInternal.cpp
13>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/qt/qt_static_plugins/CMakeLists.txt
5>GroupNetworkManager.cpp
5>StreamingMediaContext.cpp
10>lib_base.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_base\Debug\liblib_base.a
5>VideoStreamingPart.cpp
12>generator.cpp
12>main.cpp
12>options.cpp
12>parsed_file.cpp
5>DesktopInterface.cpp
5>VideoCapturerInterfaceImpl.cpp
5>VideoCapturerTrackSource.cpp
13>qt_static_plugins.cpp
12>processor.cpp
14>------ Build started: Project: lib_tl, Configuration: Debug Win32 ------
5>VideoCameraCapturer.cpp
5>json11.cpp
14>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_tl/CMakeLists.txt
13>external_qt_static_plugins.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\qt\qt_static_plugins\external_qt_static_plugins.dir\Debug\external_qt_static_plugins.lib
15>------ Build started: Project: codegen_emoji, Configuration: Debug Win32 ------
14>tl_basic_types.cpp
15>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/emoji/CMakeLists.txt
12>LINK : fatal error LNK1104: cannot open file 'dxguid.obj'
12>Done building project "codegen_lang.vcxproj" -- FAILED.
16>------ Build started: Project: td_lang_lang, Configuration: Debug Win32 ------
15>data.cpp
15>data_old.cpp
15>data_read.cpp
15>generator.cpp
16>Generating lang (td_lang)
16>'codegen\codegen\lang\Debug\codegen_lang.exe' is not recognized as an internal or external command,
16>operable program or batch file.
15>main.cpp
16>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: Custom build for 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\8ea97dfc0ac590daf18a9705c2385b50\lang_auto.timestamp.rule;C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\d9aa6b2cfb1cab74973420e8abb42e4f\td_lang_lang.rule;C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\CMakeLists.txt' exited with code 9009.
16>Done building project "td_lang_lang.vcxproj" -- FAILED.
17>------ Build started: Project: lib_stripe, Configuration: Debug Win32 ------
14>lib_tl.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_tl\lib_tl.dir\Debug\lib_tl.lib
15>options.cpp
17>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
15>replaces.cpp
18>------ Build started: Project: td_scheme, Configuration: Debug Win32 ------
5>lib_tgcalls.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\Debug\liblib_tgcalls.a
19>------ Build started: Project: lib_tgcalls_legacy, Configuration: Debug Win32 ------
19>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
17>cmake_pch.cxx
19>InstanceImplLegacy.cpp
15>LINK : fatal error LNK1104: cannot open file 'dxguid.obj'
15>Done building project "codegen_emoji.vcxproj" -- FAILED.
20>------ Build started: Project: lib_ui_emoji, Configuration: Debug Win32 ------
17>stripe_api_client.cpp
17>stripe_card.cpp
17>stripe_card_params.cpp
17>stripe_card_validator.cpp
17>stripe_decode.cpp
17>stripe_error.cpp
17>stripe_form_encoder.cpp
17>stripe_token.cpp
17>smartglocal_api_client.cpp
17>smartglocal_card.cpp
17>smartglocal_error.cpp
17>smartglocal_token.cpp
20>Generating emoji (lib_ui)
19>lib_tgcalls_legacy.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\Debug\liblib_tgcalls_legacy.a
20>'..\codegen\codegen\emoji\Debug\codegen_emoji.exe' is not recognized as an internal or external command,
20>operable program or batch file.
20>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: Custom build for 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\dc0968ae6903e5ed2ab7933799d23a4f\emoji.timestamp.rule;C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3ccafd3d47eae94874b2668472fde362\lib_ui_emoji.rule;C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\CMakeLists.txt' exited with code 9009.
20>Done building project "lib_ui_emoji.vcxproj" -- FAILED.
21>------ Build started: Project: lib_storage, Configuration: Debug Win32 ------
22>------ Build started: Project: codegen_style, Configuration: Debug Win32 ------
21>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_storage/CMakeLists.txt
22>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/codegen/codegen/style/CMakeLists.txt
21>cmake_pch.cxx
22>generator.cpp
22>main.cpp
22>module.cpp
22>options.cpp
17>lib_stripe.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_stripe.dir\Debug\lib_stripe.lib
23>------ Build started: Project: external_rlottie_bundled, Configuration: Debug Win32 ------
22>parsed_file.cpp
23>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/rlottie/CMakeLists.txt
22>processor.cpp
22>structure_types.cpp
23>lottieanimation.cpp
23>lottieitem.cpp
23>lottiekeypath.cpp
23>lottieloader.cpp
23>lottiemodel.cpp
21>storage_clear_legacy.cpp
21>storage_clear_legacy_win.cpp
21>storage_databases.cpp
21>storage_encrypted_file.cpp
23>lottieparser.cpp
21>storage_encryption.cpp
21>storage_cache_binlog_reader.cpp
21>storage_cache_cleaner.cpp
23>lottieproxymodel.cpp
22>LINK : fatal error LNK1104: cannot open file 'dxguid.obj'
21>storage_cache_compactor.cpp
23>v_ft_math.cpp
23>v_ft_raster.cpp
22>Done building project "codegen_style.vcxproj" -- FAILED.
21>storage_cache_database.cpp
23>v_ft_stroker.cpp
23>vregion.cpp
24>------ Build started: Project: td_ui_styles, Configuration: Debug Win32 ------
23>vbezier.cpp
23>vbitmap.cpp
21>storage_cache_database_object.cpp
21>storage_cache_types.cpp
23>vbrush.cpp
23>vcompositionfunctions.cpp
23>vdasher.cpp
23>vdebug.cpp
23>vdrawable.cpp
23>vdrawhelper.cpp
23>vdrawhelper_neon.cpp
23>vdrawhelper_sse2.cpp
23>velapsedtimer.cpp
23>vimageloader.cpp
23>vinterpolator.cpp
23>vmatrix.cpp
25>------ Build started: Project: lib_ui_styles, Configuration: Debug Win32 ------
23>vpainter.cpp
21>lib_storage.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_storage\lib_storage.dir\Debug\lib_storage.lib
23>vpath.cpp
25>Generating styles (lib_ui)
23>vpathmesure.cpp
26>------ Build started: Project: td_mtproto, Configuration: Debug Win32 ------
25>'..\codegen\codegen\style\Debug\codegen_style.exe' is not recognized as an internal or external command,
25>operable program or batch file.
23>vraster.cpp
25>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: Custom build for 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\2436fe89c260795f2d3ab8fc3398ff9e\lib_ui_style.timestamp.rule;C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3ccafd3d47eae94874b2668472fde362\lib_ui_styles.rule;C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\CMakeLists.txt' exited with code 9009.
25>Done building project "lib_ui_styles.vcxproj" -- FAILED.
27>------ Build started: Project: td_export, Configuration: Debug Win32 ------
23>vrect.cpp
23>vrle.cpp
27>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
26>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
28>------ Build started: Project: lib_ui_palette, Configuration: Debug Win32 ------
28>Generating palette (lib_ui)
28>'..\codegen\codegen\style\Debug\codegen_style.exe' is not recognized as an internal or external command,
28>operable program or batch file.
27>cmake_pch.cxx
26>cmake_pch.cxx
28>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: Custom build for 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\colors.palette;C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3ccafd3d47eae94874b2668472fde362\lib_ui_palette.rule;C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\CMakeLists.txt' exited with code 9009.
28>Done building project "lib_ui_palette.vcxproj" -- FAILED.
29>------ Build started: Project: lib_ui_autogen, Configuration: Debug Win32 ------
29>1>Automatic MOC for target lib_ui
29>
29>AutoGen error
29>-------------
29>Info error in info file
29>"C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/out/Telegram/lib_ui/CMakeFiles/lib_ui_autogen.dir/AutogenInfo.json":
29>The header file "SRC:/out/Telegram/lib_ui/gen/emoji.h" does not exist.
29>
29>The system cannot find the batch label specified - VCEnd
29>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(232,5): error MSB8066: Custom build for 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\CMakeFiles\3ccafd3d47eae94874b2668472fde362\lib_ui_autogen.rule' exited with code 1.
29>Done building project "lib_ui_autogen.vcxproj" -- FAILED.
30>------ Build started: Project: external_qr_code_generator_bundled, Configuration: Debug Win32 ------
23>external_rlottie_bundled.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\rlottie\Debug\libexternal_rlottie_bundled.a
31>------ Build started: Project: external_minizip_bundled, Configuration: Debug Win32 ------
30>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/qr_code_generator/CMakeLists.txt
31>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/minizip/CMakeLists.txt
30>qrcodegen.cpp
31>ioapi.c
31>zip.c
31>unzip.c
31>external_minizip_bundled.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\minizip\Debug\libexternal_minizip_bundled.a
32>------ Build started: Project: external_lz4_bundled, Configuration: Debug Win32 ------
32>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/lz4/CMakeLists.txt
32>lz4.c
32>lz4frame.c
32>lz4hc.c
32>xxhash.c
30>external_qr_code_generator_bundled.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\qr_code_generator\Debug\libexternal_qr_code_generator_bundled.a
32>external_lz4_bundled.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\lz4\Debug\libexternal_lz4_bundled.a
33>------ Build started: Project: lib_ui, Configuration: Debug Win32 ------
34>------ Build started: Project: lib_qr, Configuration: Debug Win32 ------
34>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_qr/CMakeLists.txt
33>1>Automatic RCC for qt_conf/win.qrc
34>qr_generate.cpp
33>2>Automatic RCC for fonts/fonts.qrc
33>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_ui/CMakeLists.txt
33>cmake_pch.cxx
34>lib_qr.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_qr\lib_qr.dir\Debug\lib_qr.lib
35>------ Build started: Project: external_hunspell_bundled, Configuration: Debug Win32 ------
35>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/hunspell/CMakeLists.txt
35>affentry.cxx
35>affixmgr.cxx
35>csutil.cxx
35>filemgr.cxx
35>hashmgr.cxx
35>hunspell.cxx
35>hunzip.cxx
27>export_api_wrap.cpp
27>export_controller.cpp
27>export_settings.cpp
27>export_data_types.cpp
35>phonet.cxx
35>replist.cxx
26>mtproto_abstract_socket.cpp
27>export_output_abstract.cpp
35>suggestmgr.cxx
26>mtproto_bound_key_creator.cpp
26>mtproto_dc_key_binder.cpp
26>mtproto_dc_key_creator.cpp
26>mtproto_dcenter.cpp
26>mtproto_domain_resolver.cpp
26>mtproto_dump_to_text.cpp
26>mtproto_received_ids_manager.cpp
26>mtproto_rsa_public_key.cpp
26>mtproto_serialized_request.cpp
26>mtproto_tcp_socket.cpp
27>export_output_file.cpp
26>mtproto_tls_socket.cpp
26>mtproto_auth_key.cpp
26>mtproto_concurrent_sender.cpp
26>mtproto_config.cpp
27>export_output_html.cpp
26>mtproto_dc_options.cpp
27>export_output_json.cpp
26>mtproto_dh_utils.cpp
26>mtproto_proxy_data.cpp
26>mtproto_response.cpp
33>mocs_compilation_Debug.cpp
33>c1xx : fatal  error C1083: Cannot open source file: 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\lib_ui_autogen\mocs_compilation_Debug.cpp': No such file or directory
33>palette.cpp
33>c1xx : fatal  error C1083: Cannot open source file: 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\palette.cpp': No such file or directory
33>style_basic.cpp
33>c1xx : fatal  error C1083: Cannot open source file: 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\style_basic.cpp': No such file or directory
33>style_layers.cpp
33>c1xx : fatal  error C1083: Cannot open source file: 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\style_layers.cpp': No such file or directory
33>style_widgets.cpp
35>external_hunspell_bundled.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\hunspell\Debug\libexternal_hunspell_bundled.a
33>c1xx : fatal  error C1083: Cannot open source file: 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\styles\style_widgets.cpp': No such file or directory
33>emoji.cpp
33>c1xx : fatal  error C1083: Cannot open source file: 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\emoji.cpp': No such file or directory
33>emoji_suggestions_data.cpp
33>c1xx : fatal  error C1083: Cannot open source file: 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ui\gen\emoji_suggestions_data.cpp': No such file or directory
33>dpr_icon.cpp
33>animation_value.cpp
33>animations.cpp
33>cross_animation.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/cross_animation.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\cross_animation.cpp)
33>cross_line.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/cross_line.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\cross_line.cpp)
33>fade_animation.cpp
27>export_output_stats.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\animation_value.cpp)
33>frame_generator.cpp
33>gradient.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/fade_animation.h(11,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\fade_animation.cpp)
33>numbers_animation.cpp
33>panel_animation.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/panel_animation.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\panel_animation.cpp)
33>path_shift_gradient.cpp
33>radial_animation.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\radial_animation.cpp)
33>ripple_animation.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/ripple_animation.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\ripple_animation.cpp)
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\numbers_animation.cpp)
33>round_area_with_shadow.cpp
33>show_animation.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\round_area_with_shadow.cpp)
33>slide_animation.cpp
33>spoiler_mess.cpp
33>gl_detection.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\show_animation.cpp(11,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory
33>gl_image.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\effects\spoiler_mess.cpp)
33>gl_math.cpp
33>gl_primitives.cpp
33>gl_shader.cpp
36>------ Build started: Project: external_cld3_bundled_task_spec.proto, Configuration: Debug Win32 ------
26>td_mtproto.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\td_mtproto.dir\Debug\td_mtproto.lib
33>gl_surface.cpp
33>gl_window.cpp
33>image_prepare.cpp
33>box_content.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\image\image_prepare.cpp)
33>box_layer_widget.cpp
36>Generating protobuf task_spec.proto (external_cld3_bundled)
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\gl\gl_surface.cpp)
33>generic_box.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\box_content.cpp)
33>layer_manager.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\box_layer_widget.cpp)
33>layer_widget.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\generic_box.cpp)
33>show.cpp
37>------ Build started: Project: external_cld3_bundled_sentence.proto, Configuration: Debug Win32 ------
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\layers\layer_widget.cpp)
33>arcs.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/paint/arcs.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\arcs.cpp)
33>blob.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\blob.cpp)
33>blobs.cpp
36>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\blobs.cpp)
33>blobs_linear.cpp
33>ui_window_shadow_win.cpp
33>ui_window_title_win.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\paint\blobs_linear.cpp)
33>ui_window_win.cpp
37>Generating protobuf sentence.proto (external_cld3_bundled)
33>ui_windows_direct_manipulation.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_window_shadow_win.cpp(12,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory
33>ui_utility_win.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_window_title_win.cpp)
37>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
33>ui_platform_window_title.cpp
38>------ Build started: Project: external_cld3_bundled_feature_extractor.proto, Configuration: Debug Win32 ------
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\ui_platform_window_title.cpp)
38>Generating protobuf feature_extractor.proto (external_cld3_bundled)
33>ui_platform_window.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/popup_menu.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_utility_win.cpp)
33>style_core.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core.cpp)
33>style_core_color.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\win\ui_window_win.cpp(19,1): fatal  error C1083: Cannot open include file: 'styles/palette.h': No such file or directory
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/style/style_core_palette.h(10,1): fatal  error C1083: Cannot open include file: 'styles/palette.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core_color.cpp)
33>style_core_custom_font.cpp
33>style_core_direction.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\platform\ui_platform_window.cpp)
33>style_core_font.cpp
33>style_core_icon.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/style/style_core_palette.h(10,1): fatal  error C1083: Cannot open include file: 'styles/palette.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core_icon.cpp)
33>style_core_palette.cpp
38>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
27>td_export.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\td_export.dir\Debug\td_export.lib
39>------ Build started: Project: Telegram_midl, Configuration: Debug Win32 ------
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/style/style_core_palette.h(10,1): fatal  error C1083: Cannot open include file: 'styles/palette.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\style\style_core_palette.cpp)
33>style_core_scale.cpp
33>style_core_types.cpp
33>style_palette_colorizer.cpp
33>custom_emoji_instance.cpp
33>text.cpp
33>text_block.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_block.cpp)
33>text_custom_emoji.cpp
33>text_entity.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text.cpp)
33>text_parser.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_parser.cpp)
33>text_renderer.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_entity.cpp)
33>text_spoiler_data.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/painter.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\custom_emoji_instance.cpp)
33>text_utilities.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_renderer.cpp)
33>text_variant.cpp
33>toast.cpp
39>Generating headers from IDLs (Telegram)
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\text\text_spoiler_data.cpp)
40>------ Build started: Project: external_cld3_bundled, Configuration: Debug Win32 ------
33>toast_manager.cpp
39>Microsoft (R) 32b/64b MIDL Compiler Version 8.01.0628
39>Copyright (c) Microsoft Corporation. All rights reserved.
33>toast_widget.cpp
39>Processing C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\platform\win\windows_quiethours.idl
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\toast\toast.cpp)
39>windows_quiethours.idl
33>box_content_divider.cpp
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\oaidl.idl
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\toast\toast_manager.cpp)
39>oaidl.idl
33>buttons.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\toast\toast_widget.cpp)
33>call_button.cpp
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\objidl.idl
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\box_content_divider.cpp(9,1): fatal  error C1083: Cannot open include file: 'styles/style_layers.h': No such file or directory
33>checkbox.cpp
33>dropdown_menu.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/dropdown_menu.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\dropdown_menu.cpp)
33>icon_button_with_text.cpp
39>objidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\unknwn.idl
39>unknwn.idl
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\buttons.cpp)
33>inner_dropdown.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/inner_dropdown.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\inner_dropdown.cpp)
33>input_fields.cpp
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared\wtypes.idl
39>wtypes.idl
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\input_fields.cpp)
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\call_button.cpp)
33>labels.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\checkbox.cpp)
33>time_part_input.cpp
33>menu.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\icon_button_with_text.cpp)
33>menu_action.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\fields\time_part_input.cpp)
33>menu_add_action_callback.cpp
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared\wtypesbase.idl
39>wtypesbase.idl
40>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/cmake/external/cld3/CMakeLists.txt
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared\basetsd.h
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_action.cpp)
33>menu_add_action_callback_factory.cpp
33>menu_common.cpp
39>basetsd.h
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared\guiddef.h
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_add_action_callback_factory.cpp)
39>guiddef.h
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu.cpp)
33>menu_item_base.cpp
33>menu_multiline_action.cpp
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\oaidl.acf
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\labels.cpp)
33>menu_separator.cpp
33>menu_toggle.cpp
40>feature_extractor.pb.cc
39>oaidl.acf
40>sentence.pb.cc
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_toggle.cpp)
40>task_spec.pb.cc
40>base.cc
33>elastic_scroll.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_multiline_action.cpp)
33>popup_menu.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_item_base.cpp)
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/popup_menu.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\popup_menu.cpp)
33>rp_window.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\menu\menu_separator.cpp)
33>separate_panel.cpp
33>scroll_area.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/scroll_area.h(13,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\elastic_scroll.cpp)
33>side_bar_button.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\separate_panel.cpp)
39>Microsoft (R) 32b/64b MIDL Compiler Version 8.01.0628
39>Copyright (c) Microsoft Corporation. All rights reserved.
33>shadow.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/scroll_area.h(13,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\scroll_area.cpp)
39>Processing C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\platform\win\windows_toastactivator.idl
33>time_input.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\side_bar_button.cpp)
33>tooltip.cpp
33>fade_wrap.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\shadow.cpp(10,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory
39>windows_toastactivator.idl
40>embedding_feature_extractor.cc
33>follow_slide_wrap.cpp
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\oaidl.idl
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/fade_animation.h(11,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\fade_wrap.cpp)
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\tooltip.cpp)
33>padding_wrap.cpp
33>slide_wrap.cpp
39>oaidl.idl
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\follow_slide_wrap.cpp(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\widgets\time_input.cpp)
33>vertical_layout.cpp
40>embedding_network.cc
33>vertical_layout_reorder.cpp
40>feature_extractor.cc
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\objidl.idl
40>feature_types.cc
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\slide_wrap.cpp(9,1): fatal  error C1083: Cannot open include file: 'styles/style_basic.h': No such file or directory
33>abstract_button.cpp
39>objidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\unknwn.idl
33>animated_icon.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/widgets/scroll_area.h(13,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\wrap\vertical_layout_reorder.cpp)
33>basic_click_handlers.cpp
39>unknwn.idl
33>cached_special_layer_shadow_corners.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/effects/ripple_animation.h(9,1): fatal  error C1083: Cannot open include file: 'styles/style_widgets.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\cached_special_layer_shadow_corners.cpp)
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared\wtypes.idl
33>click_handler.cpp
40>fml_parser.cc
33>delayed_activation.cpp
39>wtypes.idl
33>dragging_scroll_manager.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\basic_click_handlers.cpp)
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared\wtypesbase.idl
33>emoji_config.cpp
40>language_identifier_features.cc
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\emoji_config.cpp)
33>inactive_press.cpp
39>wtypesbase.idl
33>integration.cpp
33>main_queue_processor.cpp
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared\basetsd.h
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui\integration.cpp)
33>ph.cpp
40>lang_id_nn_params.cc
40>nnet_language_identifier.cc
39>basetsd.h
33>round_rect.cpp
33>rp_widget.cpp
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared\guiddef.h
33>ui_utility.cpp
33>emoji_suggestions.cpp
33>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\emoji_suggestions\emoji_suggestions.cpp(10,1): fatal  error C1083: Cannot open include file: 'emoji_suggestions_data.h': No such file or directory
39>guiddef.h
33>qrc_win.cpp
33>qrc_fonts.cpp
40>registry.cc
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\ocidl.idl
40>relevant_script_feature.cc
39>ocidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\oleidl.idl
40>sentence_features.cc
40>task_context.cc
39>oleidl.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\servprov.idl
39>servprov.idl
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\urlmon.idl
39>urlmon.idl
40>task_context_params.cc
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\msxml.idl
39>msxml.idl
40>unicodetext.cc
40>utils.cc
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\oaidl.acf
40>workspace.cc
39>oaidl.acf
40>generated_entities.cc
39>Processing C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\ocidl.acf
40>getonescriptspan.cc
39>ocidl.acf
40>utf8statetable.cc
40>offsetmap.cc
40>text_processing.cc
40>fixunicodevalue.cc
39>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
33>Done building project "lib_ui.vcxproj" -- FAILED.
41>------ Build started: Project: td_lang, Configuration: Debug Win32 ------
42>------ Build started: Project: lib_webview, Configuration: Debug Win32 ------
43>------ Build started: Project: lib_ffmpeg, Configuration: Debug Win32 ------
40>external_cld3_bundled.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\cmake\external\cld3\Debug\libexternal_cld3_bundled.a
44>------ Build started: Project: lib_spellcheck, Configuration: Debug Win32 ------
41>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
43>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
42>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_webview/CMakeLists.txt
41>cmake_pch.cxx
43>ffmpeg_frame_generator.cpp
42>webview_dialog.cpp
43>ffmpeg_utility.cpp
44>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_spellcheck/CMakeLists.txt
42>webview_embed.cpp
42>webview_win.cpp
42>webview_windows_edge_chromium.cpp
42>webview_windows_edge_html.cpp
44>cmake_pch.cxx
42>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_webview\webview\webview_dialog.cpp)
41>lang_auto.cpp
41>c1xx : fatal  error C1083: Cannot open source file: 'C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\gen\lang_auto.cpp': No such file or directory
41>lang_file_parser.cpp
41>lang_keys.cpp
41>lang_tag.cpp
41>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang/lang_keys.h(10,1): fatal  error C1083: Cannot open include file: 'lang_auto.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang\lang_file_parser.cpp)
41>lang_text_entity.cpp
41>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang/lang_keys.h(10,1): fatal  error C1083: Cannot open include file: 'lang_auto.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang\lang_keys.cpp)
41>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang/lang_keys.h(10,1): fatal  error C1083: Cannot open include file: 'lang_auto.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\SourceFiles\lang\lang_tag.cpp)
43>lib_ffmpeg.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_ffmpeg.dir\Debug\lib_ffmpeg.lib
41>Done building project "td_lang.vcxproj" -- FAILED.
45>------ Build started: Project: lib_webrtc, Configuration: Debug Win32 ------
46>------ Build started: Project: lib_lottie, Configuration: Debug Win32 ------
45>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_webrtc/CMakeLists.txt
46>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/lib_lottie/CMakeLists.txt
45>cmake_pch.cxx
46>lottie_frame_provider_direct.cpp
46>lottie_frame_renderer.cpp
46>lottie_animation.cpp
46>lottie_common.cpp
44>spellcheck_win.cpp
44>language_cld3.cpp
44>hunspell_controller.cpp
44>spellcheck_utils.cpp
46>lottie_frame_generator.cpp
42>Done building project "lib_webview.vcxproj" -- FAILED.
46>lottie_multi_player.cpp
46>lottie_single_player.cpp
46>lottie_icon.cpp
44>spelling_highlighter.cpp
44>spelling_highlighter_helper.cpp
44>spellcheck_value.cpp
45>webrtc_audio_input_tester.cpp
45>webrtc_create_adm.cpp
45>webrtc_media_devices.cpp
45>webrtc_video_track.cpp
46>lottie_cache.cpp
44>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_spellcheck\spellcheck\spelling_highlighter.cpp)
44>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory (compiling source file C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_spellcheck\spellcheck\spelling_highlighter_helper.cpp)
45>webrtc_openal_adm.cpp
45>webrtc_loopback_adm_win.cpp
46>lottie_cache_frame_storage.cpp
46>lottie_frame_provider_cached.cpp
46>lottie_frame_provider_cached_multi.cpp
46>lottie_frame_provider_shared.cpp
44>Done building project "lib_spellcheck.vcxproj" -- FAILED.
45>lib_webrtc.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_webrtc\lib_webrtc.dir\Debug\lib_webrtc.lib
46>lib_lottie.vcxproj -> C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\out\Telegram\lib_lottie\lib_lottie.dir\Debug\lib_lottie.lib
47>------ Build started: Project: td_ui, Configuration: Debug Win32 ------
47>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
47>cmake_pch.cxx
47>C:\Users\48242\Desktop\TBuild\BuildPath\tdesktop\Telegram\lib_ui\ui/emoji_config.h(12,1): fatal  error C1083: Cannot open include file: 'emoji.h': No such file or directory
47>Done building project "td_ui.vcxproj" -- FAILED.
48>------ Build started: Project: Telegram_autogen, Configuration: Debug Win32 ------
48>1>Automatic MOC for target Telegram
48>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
49>------ Build started: Project: Telegram, Configuration: Debug Win32 ------
49>2>Automatic RCC for Resources/qrc/emoji_4.qrc
49>2>Automatic RCC for Resources/qrc/emoji_5.qrc
49>2>Automatic RCC for Resources/qrc/emoji_6.qrc
49>1>Automatic RCC for Resources/qrc/emoji_1.qrc
49>1>Automatic RCC for Resources/qrc/emoji_2.qrc
49>1>Automatic RCC for Resources/qrc/emoji_3.qrc
49>3>Automatic RCC for Resources/qrc/emoji_7.qrc
49>3>Automatic RCC for Resources/qrc/emoji_preview.qrc
49>3>Automatic RCC for Resources/qrc/telegram/animations.qrc
49>4>Automatic RCC for Resources/qrc/telegram/export.qrc
49>4>Automatic RCC for Resources/qrc/telegram/telegram.qrc
49>4>Automatic RCC for Resources/qrc/telegram/sounds.qrc
49>Building Custom Rule C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/CMakeLists.txt
49>cmake_pch.cxx
49>C:/Users/48242/Desktop/TBuild/BuildPath/tdesktop/Telegram/SourceFiles/stdafx.h(129,1): fatal  error C1083: Cannot open include file: 'styles/palette.h': No such file or directory
49>Done building project "Telegram.vcxproj" -- FAILED.
========== Build: 34 succeeded, 15 failed, 0 up-to-date, 0 skipped ==========
========== Build started at 3:25 PM and took 03:11.859 minutes ==========


<pre>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,9 +19,11 @@ include(cmake/validate_special_target.cmake)
 include(cmake/version.cmake)
 desktop_app_parse_version(Telegram/build/version)
 
-set(project_langs ASM C CXX)
+set(project_langs C CXX)
 if (APPLE)
     list(APPEND project_langs OBJC OBJCXX)
+elseif (LINUX)
+    list(APPEND project_langs ASM)
 endif()
 
 project(Telegram

